\documentclass{article}
\usepackage{graphicx}
\usepackage{longtable}
\usepackage{booktabs}
\usepackage{multirow}
\usepackage{array}
\usepackage{float}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{hyperref}
\usepackage{geometry}
\usepackage{tabularx}
\geometry{a4paper, top=2cm, bottom=2cm, left=1cm, right=1cm}
\begin{document}
\title{Annexe}
\maketitle
\tableofcontents
\newpage
\section{Q3.1 Top 10 up-regulated transcripts without adjustement for confondant effect}
\subsection{Cluster 0 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210156.1 & 2.5397 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 2.3406 & 0.0000 \\ 
  3 & ENSG00000124159.15 & 1.8320 & 0.0000 \\ 
  4 & ENSG00000223609.7 & 1.8241 & 0.0000 \\ 
  5 & ENSG00000210140.1 & 1.6348 & 0.0000 \\ 
  6 & ENSG00000202538.1 & 1.5935 & 0.0000 \\ 
  7 & ENSG00000118972.1 & 1.5324 & 0.0000 \\ 
  8 & ENSG00000086506.2 & 1.4671 & 0.0000 \\ 
  9 & ENSG00000158578.18 & 1.4460 & 0.0000 \\ 
  10 & ENSG00000206172.8 & 1.4459 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 0} 
\label{tab:q3_1_0}
\end{table}
\subsection{Cluster 1 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210156.1 & 2.2671 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 1.8488 & 0.0000 \\ 
  3 & ENSG00000281383.1 & 1.4370 & 0.0000 \\ 
  4 & ENSG00000202538.1 & 1.3242 & 0.0000 \\ 
  5 & ENSG00000164821.4 & 1.2449 & 0.0000 \\ 
  6 & ENSG00000259384.6 & 1.2218 & 0.0000 \\ 
  7 & ENSG00000137392.9 & 1.1493 & 0.0000 \\ 
  8 & ENSG00000172232.9 & 1.1267 & 0.0000 \\ 
  9 & ENSG00000110245.11 & 1.0570 & 0.0000 \\ 
  10 & ENSG00000279691.1 & 1.0513 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 1} 
\label{tab:q3_1_1}
\end{table}
\subsection{Cluster 2 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210156.1 & 2.1983 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 1.9498 & 0.0000 \\ 
  3 & ENSG00000210140.1 & 1.5320 & 0.0000 \\ 
  4 & ENSG00000281383.1 & 1.4409 & 0.0000 \\ 
  5 & ENSG00000262902.1 & 1.1566 & 0.0000 \\ 
  6 & ENSG00000202538.1 & 1.1543 & 0.0000 \\ 
  7 & ENSG00000179751.6 & 1.1436 & 0.0000 \\ 
  8 & ENSG00000137392.9 & 1.0442 & 0.0000 \\ 
  9 & ENSG00000168928.12 & 1.0308 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 2} 
\label{tab:q3_1_2}
\end{table}
\subsection{Cluster 3 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210154.1 & 2.0926 & 0.0000 \\ 
  2 & ENSG00000202538.1 & 1.5827 & 0.0000 \\ 
  3 & ENSG00000200879.1 & 1.4256 & 0.0000 \\ 
  4 & ENSG00000259710.1 & 1.3485 & 0.0000 \\ 
  5 & ENSG00000277581.1 & 1.3335 & 0.0000 \\ 
  6 & ENSG00000267703.1 & 1.3278 & 0.0000 \\ 
  7 & ENSG00000260799.1 & 1.3239 & 0.0000 \\ 
  8 & ENSG00000169429.10 & 1.3133 & 0.0000 \\ 
  9 & ENSG00000237070.1 & 1.2797 & 0.0000 \\ 
  10 & ENSG00000118972.1 & 1.2757 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 3} 
\label{tab:q3_1_3}
\end{table}
\subsection{Cluster 4 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000158578.18 & 2.9068 & 0.0000 \\ 
  2 & ENSG00000213934.6 & 2.9066 & 0.0000 \\ 
  3 & ENSG00000260592.1 & 2.8063 & 0.0000 \\ 
  4 & ENSG00000169877.9 & 2.7608 & 0.0000 \\ 
  5 & ENSG00000188536.12 & 2.6596 & 0.0000 \\ 
  6 & ENSG00000196565.13 & 2.4875 & 0.0000 \\ 
  7 & ENSG00000215182.8 & 2.4850 & 0.0000 \\ 
  8 & ENSG00000206177.6 & 2.4165 & 0.0000 \\ 
  9 & ENSG00000261039.2 & 2.3931 & 0.0000 \\ 
  10 & ENSG00000095752.6 & 2.3751 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 4} 
\label{tab:q3_1_4}
\end{table}
\subsection{Cluster 5 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210156.1 & 2.3618 & 0.0000 \\ 
  2 & ENSG00000210154.1 & 2.2689 & 0.0000 \\ 
  3 & ENSG00000281181.1 & 1.9596 & 0.0000 \\ 
  4 & ENSG00000108342.12 & 1.6685 & 0.0000 \\ 
  5 & ENSG00000281383.1 & 1.4671 & 0.0000 \\ 
  6 & ENSG00000118972.1 & 1.4470 & 0.0000 \\ 
  7 & ENSG00000202538.1 & 1.4464 & 0.0000 \\ 
  8 & ENSG00000210140.1 & 1.4281 & 0.0000 \\ 
  9 & ENSG00000164821.4 & 1.2934 & 0.0000 \\ 
  10 & ENSG00000172232.9 & 1.2923 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 5} 
\label{tab:q3_1_5}
\end{table}
\subsection{Cluster 6 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000248713.1 & 1.2854 & 0.0000 \\ 
  2 & ENSG00000073737.16 & 1.0690 & 0.0000 \\ 
  3 & ENSG00000223617.1 & 1.0666 & 0.0000 \\ 
  4 & ENSG00000231013.1 & 1.0187 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 6} 
\label{tab:q3_1_6}
\end{table}
\subsection{Cluster 7 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000281181.1 & 2.2393 & 0.0000 \\ 
  2 & ENSG00000202538.1 & 2.1307 & 0.0000 \\ 
  3 & ENSG00000174697.4 & 2.0061 & 0.0000 \\ 
  4 & ENSG00000281383.1 & 1.8399 & 0.0000 \\ 
  5 & ENSG00000181092.9 & 1.6299 & 0.0000 \\ 
  6 & ENSG00000254211.5 & 1.6232 & 0.0000 \\ 
  7 & ENSG00000124159.15 & 1.5154 & 0.0008 \\ 
  8 & ENSG00000130876.11 & 1.4476 & 0.0000 \\ 
  9 & ENSG00000187288.10 & 1.4213 & 0.0000 \\ 
  10 & ENSG00000166819.11 & 1.4000 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 7} 
\label{tab:q3_1_7}
\end{table}
\subsection{Cluster 8 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000110680.12 & 2.7624 & 0.0000 \\ 
  2 & ENSG00000229807.10 & 2.6810 & 0.0000 \\ 
  3 & ENSG00000270641.1 & 2.3376 & 0.0008 \\ 
  4 & ENSG00000134240.11 & 2.2957 & 0.0000 \\ 
  5 & ENSG00000180053.7 & 2.2034 & 0.0000 \\ 
  6 & ENSG00000283378.1 & 2.0152 & 0.0000 \\ 
  7 & ENSG00000279431.1 & 1.8317 & 0.0001 \\ 
  8 & ENSG00000178233.17 & 1.7337 & 0.0000 \\ 
  9 & ENSG00000257647.1 & 1.6973 & 0.0000 \\ 
  10 & ENSG00000073737.16 & 1.6461 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 8} 
\label{tab:q3_1_8}
\end{table}
\subsection{Cluster 9 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210154.1 & 2.3534 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 2.0067 & 0.0000 \\ 
  3 & ENSG00000140465.13 & 1.5961 & 0.0000 \\ 
  4 & ENSG00000179914.4 & 1.4568 & 0.0000 \\ 
  5 & ENSG00000202538.1 & 1.4368 & 0.0000 \\ 
  6 & ENSG00000210140.1 & 1.4310 & 0.0000 \\ 
  7 & ENSG00000102854.15 & 1.2138 & 0.0000 \\ 
  8 & ENSG00000086506.2 & 1.1990 & 0.0000 \\ 
  9 & ENSG00000169347.16 & 1.1893 & 0.0000 \\ 
  10 & ENSG00000108342.12 & 1.1760 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 9} 
\label{tab:q3_1_9}
\end{table}
\subsection{Cluster 10 }
No significant transcript
\subsection{Cluster 11 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210191.1 & 3.8454 & 0.0000 \\ 
  2 & ENSG00000223609.7 & 3.6328 & 0.0000 \\ 
  3 & ENSG00000086506.2 & 3.2895 & 0.0000 \\ 
  4 & ENSG00000179914.4 & 2.9044 & 0.0000 \\ 
  5 & ENSG00000281181.1 & 2.6384 & 0.0000 \\ 
  6 & ENSG00000158578.18 & 2.5955 & 0.0000 \\ 
  7 & ENSG00000260592.1 & 2.5845 & 0.0000 \\ 
  8 & ENSG00000213934.6 & 2.5189 & 0.0000 \\ 
  9 & ENSG00000176840.11 & 2.5185 & 0.0000 \\ 
  10 & ENSG00000283907.1 & 2.4638 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 11} 
\label{tab:q3_1_11}
\end{table}
\subsection{Cluster 12 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000281181.1 & 2.3840 & 0.0000 \\ 
  2 & ENSG00000202538.1 & 1.8674 & 0.0000 \\ 
  3 & ENSG00000151365.2 & 1.5169 & 0.0000 \\ 
  4 & ENSG00000146678.9 & 1.4277 & 0.0000 \\ 
  5 & ENSG00000174697.4 & 1.3273 & 0.0000 \\ 
  6 & ENSG00000006128.11 & 1.2608 & 0.0000 \\ 
  7 & ENSG00000128510.10 & 1.2010 & 0.0000 \\ 
  8 & ENSG00000283907.1 & 1.1917 & 0.0000 \\ 
  9 & ENSG00000215182.8 & 1.1266 & 0.0000 \\ 
  10 & ENSG00000211660.3 & 1.1219 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 12} 
\label{tab:q3_1_12}
\end{table}
\subsection{Cluster 13 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000140465.13 & 2.4286 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 2.0570 & 0.0000 \\ 
  3 & ENSG00000118972.1 & 1.7030 & 0.0000 \\ 
  4 & ENSG00000281383.1 & 1.6919 & 0.0000 \\ 
  5 & ENSG00000283907.1 & 1.5013 & 0.0000 \\ 
  6 & ENSG00000283209.1 & 1.4903 & 0.0000 \\ 
  7 & ENSG00000004939.13 & 1.4224 & 0.0000 \\ 
  8 & ENSG00000272799.1 & 1.4140 & 0.0000 \\ 
  9 & ENSG00000279691.1 & 1.4105 & 0.0000 \\ 
  10 & ENSG00000262902.1 & 1.4102 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 13} 
\label{tab:q3_1_13}
\end{table}
\subsection{Cluster 14 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000281181.1 & 1.7916 & 0.0007 \\ 
  2 & ENSG00000137392.9 & 1.7267 & 0.0010 \\ 
  3 & ENSG00000202538.1 & 1.5840 & 0.0002 \\ 
  4 & ENSG00000170890.13 & 1.3837 & 0.0015 \\ 
  5 & ENSG00000170827.9 & 1.2792 & 0.0005 \\ 
  6 & ENSG00000010438.16 & 1.1808 & 0.0003 \\ 
  7 & ENSG00000162438.11 & 1.1525 & 0.0005 \\ 
  8 & ENSG00000073737.16 & 1.1220 & 0.0001 \\ 
  9 & ENSG00000250746.1 & 1.0015 & 0.0001 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 14} 
\label{tab:q3_1_14}
\end{table}
\subsection{Cluster 15 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210154.1 & 2.2148 & 0.0000 \\ 
  2 & ENSG00000140465.13 & 2.0216 & 0.0000 \\ 
  3 & ENSG00000281181.1 & 1.7257 & 0.0000 \\ 
  4 & ENSG00000086506.2 & 1.4677 & 0.0000 \\ 
  5 & ENSG00000179914.4 & 1.3641 & 0.0000 \\ 
  6 & ENSG00000202538.1 & 1.3487 & 0.0000 \\ 
  7 & ENSG00000169429.10 & 1.2953 & 0.0000 \\ 
  8 & ENSG00000266378.1 & 1.2936 & 0.0000 \\ 
  9 & ENSG00000210195.2 & 1.2886 & 0.0000 \\ 
  10 & ENSG00000169245.5 & 1.2805 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 15} 
\label{tab:q3_1_15}
\end{table}
\subsection{Cluster 16 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210191.1 & 2.8695 & 0.0000 \\ 
  2 & ENSG00000210156.1 & 2.6294 & 0.0000 \\ 
  3 & ENSG00000086506.2 & 2.5888 & 0.0000 \\ 
  4 & ENSG00000158578.18 & 2.2732 & 0.0000 \\ 
  5 & ENSG00000206172.8 & 2.2046 & 0.0000 \\ 
  6 & ENSG00000004939.13 & 2.1860 & 0.0000 \\ 
  7 & ENSG00000260592.1 & 2.1850 & 0.0000 \\ 
  8 & ENSG00000169877.9 & 2.1849 & 0.0000 \\ 
  9 & ENSG00000213934.6 & 2.1419 & 0.0000 \\ 
  10 & ENSG00000188536.12 & 2.1169 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 16} 
\label{tab:q3_1_16}
\end{table}
\subsection{Cluster 17 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000281181.1 & 5.1242 & 0.0000 \\ 
  2 & ENSG00000281383.1 & 3.7459 & 0.0000 \\ 
  3 & ENSG00000202538.1 & 3.3581 & 0.0000 \\ 
  4 & ENSG00000124159.15 & 2.5998 & 0.0000 \\ 
  5 & ENSG00000086506.2 & 2.4133 & 0.0000 \\ 
  6 & ENSG00000105664.10 & 2.2957 & 0.0000 \\ 
  7 & ENSG00000223609.7 & 2.2255 & 0.0002 \\ 
  8 & ENSG00000171951.4 & 1.9575 & 0.0000 \\ 
  9 & ENSG00000095752.6 & 1.8639 & 0.0000 \\ 
  10 & ENSG00000149948.13 & 1.7847 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 17} 
\label{tab:q3_1_17}
\end{table}
\subsection{Cluster 18 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000128965.11 & 1.3977 & 0.0000 \\ 
  2 & ENSG00000103888.16 & 1.1822 & 0.0002 \\ 
  3 & ENSG00000248923.1 & 1.1284 & 0.0012 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 18} 
\label{tab:q3_1_18}
\end{table}
\subsection{Cluster 19 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000172179.11 & 2.4814 & 0.0000 \\ 
  2 & ENSG00000259384.6 & 2.3679 & 0.0000 \\ 
  3 & ENSG00000176840.11 & 2.1539 & 0.0000 \\ 
  4 & ENSG00000229859.9 & 1.8912 & 0.0001 \\ 
  5 & ENSG00000104826.12 & 1.7859 & 0.0000 \\ 
  6 & ENSG00000140465.13 & 1.7765 & 0.0000 \\ 
  7 & ENSG00000232810.3 & 1.7700 & 0.0000 \\ 
  8 & ENSG00000262902.1 & 1.7638 & 0.0000 \\ 
  9 & ENSG00000172232.9 & 1.7377 & 0.0000 \\ 
  10 & ENSG00000170827.9 & 1.7109 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 19} 
\label{tab:q3_1_19}
\end{table}
\subsection{Cluster 20 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000202538.1 & 1.5443 & 0.0000 \\ 
  2 & ENSG00000279431.1 & 1.2265 & 0.0000 \\ 
  3 & ENSG00000283265.1 & 1.1354 & 0.0000 \\ 
  4 & ENSG00000180053.7 & 1.1228 & 0.0000 \\ 
  5 & ENSG00000108759.3 & 1.1075 & 0.0001 \\ 
  6 & ENSG00000214940.8 & 1.1061 & 0.0002 \\ 
  7 & ENSG00000257647.1 & 1.0312 & 0.0000 \\ 
  8 & ENSG00000229807.10 & 1.0274 & 0.0006 \\ 
  9 & ENSG00000122735.15 & 1.0225 & 0.0000 \\ 
  10 & ENSG00000235832.2 & 1.0216 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 20} 
\label{tab:q3_1_20}
\end{table}
\subsection{Cluster 21 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210154.1 & 2.4511 & 0.0000 \\ 
  2 & ENSG00000210156.1 & 2.3969 & 0.0000 \\ 
  3 & ENSG00000281181.1 & 2.0040 & 0.0000 \\ 
  4 & ENSG00000140465.13 & 1.6920 & 0.0000 \\ 
  5 & ENSG00000118972.1 & 1.6193 & 0.0000 \\ 
  6 & ENSG00000086506.2 & 1.4118 & 0.0000 \\ 
  7 & ENSG00000169877.9 & 1.3559 & 0.0000 \\ 
  8 & ENSG00000169245.5 & 1.3555 & 0.0000 \\ 
  9 & ENSG00000004939.13 & 1.3089 & 0.0000 \\ 
  10 & ENSG00000206172.8 & 1.3087 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 21} 
\label{tab:q3_1_21}
\end{table}
\subsection{Cluster 22 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210154.1 & 2.5675 & 0.0000 \\ 
  2 & ENSG00000086506.2 & 2.0612 & 0.0000 \\ 
  3 & ENSG00000202538.1 & 1.9462 & 0.0000 \\ 
  4 & ENSG00000239839.6 & 1.6974 & 0.0000 \\ 
  5 & ENSG00000223609.7 & 1.6815 & 0.0000 \\ 
  6 & ENSG00000004939.13 & 1.6800 & 0.0000 \\ 
  7 & ENSG00000213934.6 & 1.6040 & 0.0000 \\ 
  8 & ENSG00000200879.1 & 1.6010 & 0.0000 \\ 
  9 & ENSG00000149516.13 & 1.5871 & 0.0000 \\ 
  10 & ENSG00000164821.4 & 1.5807 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 22} 
\label{tab:q3_1_22}
\end{table}
\subsection{Cluster 23 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000229828.2 & 1.2181 & 0.0000 \\ 
  2 & ENSG00000279431.1 & 1.2089 & 0.0000 \\ 
  3 & ENSG00000223617.1 & 1.0312 & 0.0000 \\ 
  4 & ENSG00000257647.1 & 1.0287 & 0.0000 \\ 
  5 & ENSG00000163082.9 & 1.0038 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 23} 
\label{tab:q3_1_23}
\end{table}
\subsection{Cluster 24 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210154.1 & 2.3879 & 0.0000 \\ 
  2 & ENSG00000210156.1 & 2.2983 & 0.0000 \\ 
  3 & ENSG00000281181.1 & 2.0785 & 0.0000 \\ 
  4 & ENSG00000281383.1 & 1.5831 & 0.0000 \\ 
  5 & ENSG00000210140.1 & 1.4668 & 0.0000 \\ 
  6 & ENSG00000202538.1 & 1.3899 & 0.0000 \\ 
  7 & ENSG00000108342.12 & 1.3539 & 0.0000 \\ 
  8 & ENSG00000181143.15 & 1.2888 & 0.0000 \\ 
  9 & ENSG00000179914.4 & 1.2834 & 0.0000 \\ 
  10 & ENSG00000137392.9 & 1.2742 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 24} 
\label{tab:q3_1_24}
\end{table}
\subsection{Cluster 25 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000279431.1 & 1.0949 & 0.0000 \\ 
  2 & ENSG00000073737.16 & 1.0196 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 25} 
\label{tab:q3_1_25}
\end{table}
\subsection{Cluster 26 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210156.1 & 2.2537 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 1.9725 & 0.0000 \\ 
  3 & ENSG00000118972.1 & 1.5372 & 0.0000 \\ 
  4 & ENSG00000210140.1 & 1.4536 & 0.0000 \\ 
  5 & ENSG00000163435.15 & 1.3491 & 0.0000 \\ 
  6 & ENSG00000140465.13 & 1.3345 & 0.0000 \\ 
  7 & ENSG00000137392.9 & 1.3111 & 0.0000 \\ 
  8 & ENSG00000202538.1 & 1.2916 & 0.0000 \\ 
  9 & ENSG00000210154.1 & 1.2621 & 0.0000 \\ 
  10 & ENSG00000168928.12 & 1.2578 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 26} 
\label{tab:q3_1_26}
\end{table}
\subsection{Cluster 27 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000281181.1 & 1.4906 & 0.0000 \\ 
  2 & ENSG00000202538.1 & 1.3269 & 0.0000 \\ 
  3 & ENSG00000279431.1 & 1.0137 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 27} 
\label{tab:q3_1_27}
\end{table}
\subsection{Cluster 28 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210156.1 & 2.2717 & 0.0000 \\ 
  2 & ENSG00000281181.1 & 2.0990 & 0.0000 \\ 
  3 & ENSG00000223609.7 & 2.0563 & 0.0000 \\ 
  4 & ENSG00000169429.10 & 1.7391 & 0.0000 \\ 
  5 & ENSG00000281383.1 & 1.6064 & 0.0000 \\ 
  6 & ENSG00000158578.18 & 1.5576 & 0.0000 \\ 
  7 & ENSG00000202538.1 & 1.5280 & 0.0000 \\ 
  8 & ENSG00000169245.5 & 1.5146 & 0.0000 \\ 
  9 & ENSG00000118972.1 & 1.5026 & 0.0000 \\ 
  10 & ENSG00000086506.2 & 1.5020 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 28} 
\label{tab:q3_1_28}
\end{table}
\subsection{Cluster 29 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000223609.7 & 3.3655 & 0.0000 \\ 
  2 & ENSG00000202538.1 & 3.2211 & 0.0000 \\ 
  3 & ENSG00000124159.15 & 2.6905 & 0.0000 \\ 
  4 & ENSG00000105664.10 & 2.5587 & 0.0000 \\ 
  5 & ENSG00000158578.18 & 2.4258 & 0.0000 \\ 
  6 & ENSG00000086506.2 & 2.3888 & 0.0000 \\ 
  7 & ENSG00000261039.2 & 2.3035 & 0.0000 \\ 
  8 & ENSG00000171951.4 & 2.1793 & 0.0000 \\ 
  9 & ENSG00000095752.6 & 2.1411 & 0.0000 \\ 
  10 & ENSG00000106483.11 & 2.1102 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 29} 
\label{tab:q3_1_29}
\end{table}
\subsection{Cluster 30 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210195.2 & 1.7112 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 30} 
\label{tab:q3_1_30}
\end{table}
\subsection{Cluster 31 }
No significant transcript
\section{Q3.1 Top 10 up-regulated transcript with adjustement for confondant effect}
\subsection{Cluster 0 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000223609.7 & 1.1369 & 0.0002 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 0 with adjustement for confondant effect} 
\label{tab:q3_1_conf_0}
\end{table}
\subsection{Cluster 1 }
No significant transcript
\subsection{Cluster 2 }
No significant transcript
\subsection{Cluster 3 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000270640.1 & 1.8560 & 0.0000 \\ 
  2 & ENSG00000215182.8 & 1.3792 & 0.0000 \\ 
  3 & ENSG00000264204.2 & 1.1567 & 0.0000 \\ 
  4 & ENSG00000159261.10 & 1.1012 & 0.0015 \\ 
  5 & ENSG00000200879.1 & 1.0460 & 0.0002 \\ 
  6 & ENSG00000095752.6 & 1.0370 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 3 with adjustement for confondant effect} 
\label{tab:q3_1_conf_3}
\end{table}
\subsection{Cluster 4 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000171951.4 & 2.6749 & 0.0000 \\ 
  2 & ENSG00000159261.10 & 2.5414 & 0.0001 \\ 
  3 & ENSG00000223609.7 & 2.5085 & 0.0000 \\ 
  4 & ENSG00000095752.6 & 2.2721 & 0.0000 \\ 
  5 & ENSG00000261039.2 & 2.0630 & 0.0001 \\ 
  6 & ENSG00000268941.2 & 2.0243 & 0.0003 \\ 
  7 & ENSG00000106541.11 & 1.9935 & 0.0002 \\ 
  8 & ENSG00000149948.13 & 1.8640 & 0.0002 \\ 
  9 & ENSG00000124159.15 & 1.7444 & 0.0007 \\ 
  10 & ENSG00000106483.11 & 1.4863 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 4 with adjustement for confondant effect} 
\label{tab:q3_1_conf_4}
\end{table}
\subsection{Cluster 5 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000163435.15 & 1.0765 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 5 with adjustement for confondant effect} 
\label{tab:q3_1_conf_5}
\end{table}
\subsection{Cluster 6 }
No significant transcript
\subsection{Cluster 7 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000174697.4 & 2.0496 & 0.0000 \\ 
  2 & ENSG00000151365.2 & 1.7741 & 0.0000 \\ 
  3 & ENSG00000181092.9 & 1.6654 & 0.0000 \\ 
  4 & ENSG00000254211.5 & 1.6275 & 0.0000 \\ 
  5 & ENSG00000187288.10 & 1.5227 & 0.0000 \\ 
  6 & ENSG00000166819.11 & 1.4918 & 0.0000 \\ 
  7 & ENSG00000130876.11 & 1.3605 & 0.0000 \\ 
  8 & ENSG00000166828.2 & 1.3040 & 0.0001 \\ 
  9 & ENSG00000133317.14 & 1.2826 & 0.0000 \\ 
  10 & ENSG00000145824.12 & 1.1655 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 7 with adjustement for confondant effect} 
\label{tab:q3_1_conf_7}
\end{table}
\subsection{Cluster 8 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000229807.10 & 2.2735 & 0.0009 \\ 
  2 & ENSG00000246375.2 & 1.7102 & 0.0000 \\ 
  3 & ENSG00000162763.3 & 1.5895 & 0.0006 \\ 
  4 & ENSG00000283378.1 & 1.5257 & 0.0010 \\ 
  5 & ENSG00000241054.1 & 1.3848 & 0.0006 \\ 
  6 & ENSG00000198734.10 & 1.2633 & 0.0001 \\ 
  7 & ENSG00000250061.5 & 1.1986 & 0.0007 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 8 with adjustement for confondant effect} 
\label{tab:q3_1_conf_8}
\end{table}
\subsection{Cluster 9 }
No significant transcript
\subsection{Cluster 10 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000219073.7 & 1.1604 & 0.0001 \\ 
  2 & ENSG00000142789.19 & 1.0556 & 0.0003 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 10 with adjustement for confondant effect} 
\label{tab:q3_1_conf_10}
\end{table}
\subsection{Cluster 11 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000277247.1 & 2.8346 & 0.0000 \\ 
  2 & ENSG00000189410.11 & 2.1225 & 0.0001 \\ 
  3 & ENSG00000223609.7 & 1.9738 & 0.0014 \\ 
  4 & ENSG00000149948.13 & 1.8667 & 0.0001 \\ 
  5 & ENSG00000086506.2 & 1.8515 & 0.0014 \\ 
  6 & ENSG00000124875.9 & 1.7872 & 0.0001 \\ 
  7 & ENSG00000239839.6 & 1.7854 & 0.0014 \\ 
  8 & ENSG00000163435.15 & 1.6569 & 0.0009 \\ 
  9 & ENSG00000229604.2 & 1.6353 & 0.0000 \\ 
  10 & ENSG00000101670.11 & 1.6247 & 0.0006 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 11 with adjustement for confondant effect} 
\label{tab:q3_1_conf_11}
\end{table}
\subsection{Cluster 12 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000151365.2 & 1.4695 & 0.0000 \\ 
  2 & ENSG00000174697.4 & 1.4452 & 0.0000 \\ 
  3 & ENSG00000202538.1 & 1.2194 & 0.0000 \\ 
  4 & ENSG00000105880.4 & 1.1673 & 0.0000 \\ 
  5 & ENSG00000254211.5 & 1.1559 & 0.0000 \\ 
  6 & ENSG00000181092.9 & 1.1122 & 0.0000 \\ 
  7 & ENSG00000146678.9 & 1.0632 & 0.0005 \\ 
  8 & ENSG00000215182.8 & 1.0605 & 0.0001 \\ 
  9 & ENSG00000006128.11 & 1.0396 & 0.0000 \\ 
  10 & ENSG00000166819.11 & 1.0377 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 12 with adjustement for confondant effect} 
\label{tab:q3_1_conf_12}
\end{table}
\subsection{Cluster 13 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000185988.11 & 1.3936 & 0.0000 \\ 
  2 & ENSG00000272799.1 & 1.1348 & 0.0000 \\ 
  3 & ENSG00000279281.1 & 1.0549 & 0.0005 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 13 with adjustement for confondant effect} 
\label{tab:q3_1_conf_13}
\end{table}
\subsection{Cluster 14 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000010438.16 & 1.1909 & 0.0003 \\ 
  2 & ENSG00000162438.11 & 1.0712 & 0.0013 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 14 with adjustement for confondant effect} 
\label{tab:q3_1_conf_14}
\end{table}
\subsection{Cluster 15 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000223609.7 & 1.2400 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 15 with adjustement for confondant effect} 
\label{tab:q3_1_conf_15}
\end{table}
\subsection{Cluster 16 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000223609.7 & 1.5988 & 0.0000 \\ 
  2 & ENSG00000159261.10 & 1.5516 & 0.0000 \\ 
  3 & ENSG00000215182.8 & 1.4767 & 0.0000 \\ 
  4 & ENSG00000237070.1 & 1.2521 & 0.0000 \\ 
  5 & ENSG00000228495.1 & 1.2404 & 0.0000 \\ 
  6 & ENSG00000169245.5 & 1.2301 & 0.0000 \\ 
  7 & ENSG00000241112.1 & 1.2216 & 0.0000 \\ 
  8 & ENSG00000206177.6 & 1.2016 & 0.0000 \\ 
  9 & ENSG00000267703.1 & 1.1947 & 0.0000 \\ 
  10 & ENSG00000086506.2 & 1.1716 & 0.0003 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 16 with adjustement for confondant effect} 
\label{tab:q3_1_conf_16}
\end{table}
\subsection{Cluster 17 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000169429.10 & 1.7999 & 0.0000 \\ 
  2 & ENSG00000149948.13 & 1.7278 & 0.0000 \\ 
  3 & ENSG00000103888.16 & 1.6323 & 0.0000 \\ 
  4 & ENSG00000124159.15 & 1.5569 & 0.0007 \\ 
  5 & ENSG00000095752.6 & 1.5507 & 0.0003 \\ 
  6 & ENSG00000281383.1 & 1.5111 & 0.0009 \\ 
  7 & ENSG00000210196.2 & 1.4883 & 0.0003 \\ 
  8 & ENSG00000171951.4 & 1.4869 & 0.0000 \\ 
  9 & ENSG00000060718.20 & 1.4805 & 0.0005 \\ 
  10 & ENSG00000166923.10 & 1.4775 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 17 with adjustement for confondant effect} 
\label{tab:q3_1_conf_17}
\end{table}
\subsection{Cluster 18 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000062038.13 & 1.3323 & 0.0002 \\ 
  2 & ENSG00000248923.1 & 1.3163 & 0.0000 \\ 
  3 & ENSG00000103888.16 & 1.2854 & 0.0001 \\ 
  4 & ENSG00000135069.13 & 1.1049 & 0.0014 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 18 with adjustement for confondant effect} 
\label{tab:q3_1_conf_18}
\end{table}
\subsection{Cluster 19 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000172179.11 & 2.2017 & 0.0001 \\ 
  2 & ENSG00000140465.13 & 1.5658 & 0.0003 \\ 
  3 & ENSG00000176840.11 & 1.5456 & 0.0014 \\ 
  4 & ENSG00000259384.6 & 1.4193 & 0.0012 \\ 
  5 & ENSG00000256713.7 & 1.3759 & 0.0003 \\ 
  6 & ENSG00000170827.9 & 1.3690 & 0.0005 \\ 
  7 & ENSG00000104826.12 & 1.2968 & 0.0004 \\ 
  8 & ENSG00000244921.2 & 1.2365 & 0.0000 \\ 
  9 & ENSG00000198744.5 & 1.2041 & 0.0002 \\ 
  10 & ENSG00000010438.16 & 1.1691 & 0.0012 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 19 with adjustement for confondant effect} 
\label{tab:q3_1_conf_19}
\end{table}
\subsection{Cluster 20 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000215182.8 & 1.4007 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 20 with adjustement for confondant effect} 
\label{tab:q3_1_conf_20}
\end{table}
\subsection{Cluster 21 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000185988.11 & 1.0874 & 0.0001 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 21 with adjustement for confondant effect} 
\label{tab:q3_1_conf_21}
\end{table}
\subsection{Cluster 22 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000239839.6 & 1.4162 & 0.0000 \\ 
  2 & ENSG00000200879.1 & 1.0568 & 0.0009 \\ 
  3 & ENSG00000237070.1 & 1.0291 & 0.0000 \\ 
  4 & ENSG00000077274.8 & 1.0056 & 0.0002 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 22 with adjustement for confondant effect} 
\label{tab:q3_1_conf_22}
\end{table}
\subsection{Cluster 23 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000174697.4 & 1.0171 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 23 with adjustement for confondant effect} 
\label{tab:q3_1_conf_23}
\end{table}
\subsection{Cluster 24 }
No significant transcript
\subsection{Cluster 25 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000214940.8 & 1.1796 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 25 with adjustement for confondant effect} 
\label{tab:q3_1_conf_25}
\end{table}
\subsection{Cluster 26 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000163435.15 & 1.0422 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 26 with adjustement for confondant effect} 
\label{tab:q3_1_conf_26}
\end{table}
\subsection{Cluster 27 }
No significant transcript
\subsection{Cluster 28 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000163435.15 & 1.1236 & 0.0000 \\ 
  2 & ENSG00000159261.10 & 1.0506 & 0.0000 \\ 
  3 & ENSG00000169429.10 & 1.0347 & 0.0000 \\ 
  4 & ENSG00000239839.6 & 1.0173 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 28 with adjustement for confondant effect} 
\label{tab:q3_1_conf_28}
\end{table}
\subsection{Cluster 29 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000183813.6 & 3.8872 & 0.0000 \\ 
  2 & ENSG00000102468.10 & 2.9228 & 0.0000 \\ 
  3 & ENSG00000257894.2 & 2.2761 & 0.0001 \\ 
  4 & ENSG00000178773.14 & 2.2342 & 0.0000 \\ 
  5 & ENSG00000261039.2 & 1.9669 & 0.0000 \\ 
  6 & ENSG00000202538.1 & 1.9596 & 0.0002 \\ 
  7 & ENSG00000167105.7 & 1.9554 & 0.0000 \\ 
  8 & ENSG00000215182.8 & 1.9496 & 0.0002 \\ 
  9 & ENSG00000095752.6 & 1.9404 & 0.0000 \\ 
  10 & ENSG00000146678.9 & 1.9214 & 0.0008 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 29 with adjustement for confondant effect} 
\label{tab:q3_1_conf_29}
\end{table}
\subsection{Cluster 30 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & gene & log2FoldChange & padj \\ 
  \hline
1 & ENSG00000210195.2 & 1.1028 & 0.0000 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated transcripts for cluster 30 with adjustement for confondant effect} 
\label{tab:q3_1_conf_30}
\end{table}
\subsection{Cluster 31 }
No significant transcript
\section{Q3.2 Top 10 up-regulated pathways without adjustement for confondant effect}
\subsection{Cluster 0 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0028 & 1.7165 \\ 
  2 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0020 & 1.6721 \\ 
  3 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0051 & 1.6072 \\ 
  4 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0027 & 1.6020 \\ 
  5 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0039 & 1.5645 \\ 
  6 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0066 & 1.5489 \\ 
  7 & REACTOME\_GRB2\_EVENTS\_IN\_ERBB2\_SIGNALING & 0.0086 & 1.4917 \\ 
  8 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0068 & 1.4024 \\ 
  9 & REACTOME\_ARACHIDONIC\_ACID\_METABOLISM & 0.0087 & 1.3710 \\ 
  10 & REACTOME\_AUTOPHAGY & 0.0053 & 1.3372 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 0} 
\label{tab:q3_2_0}
\end{table}
\subsection{Cluster 1 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0051 & 1.6022 \\ 
  2 & REACTOME\_ACTIVATION\_OF\_PPARGC1A\_PGC\_1ALPHA\_BY\_PHOSPHORYLATION & 0.0065 & 1.5865 \\ 
  3 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0061 & 1.5646 \\ 
  4 & REACTOME\_CARNITINE\_METABOLISM & 0.0069 & 1.5270 \\ 
  5 & REACTOME\_INSULIN\_RECEPTOR\_RECYCLING & 0.0098 & 1.5189 \\ 
  6 & REACTOME\_BLOOD\_GROUP\_SYSTEMS\_BIOSYNTHESIS & 0.0093 & 1.5058 \\ 
  7 & REACTOME\_SIGNALING\_BY\_NTRK3\_TRKC & 0.0101 & 1.4768 \\ 
  8 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0045 & 1.4231 \\ 
  9 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0095 & 1.4120 \\ 
  10 & REACTOME\_SIGNALING\_BY\_INSULIN\_RECEPTOR & 0.0072 & 1.3514 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 1} 
\label{tab:q3_2_1}
\end{table}
\subsection{Cluster 10 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ECM\_PROTEOGLYCANS & 0.0003 & 1.8699 \\ 
  2 & REACTOME\_ESR\_MEDIATED\_SIGNALING & 0.0003 & 1.6754 \\ 
  3 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0003 & 1.4632 \\ 
  4 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0003 & 1.4534 \\ 
  5 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0000 & 1.4521 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0001 & 1.4417 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0001 & 1.4093 \\ 
  8 & REACTOME\_CELL\_CYCLE & 0.0003 & 1.3543 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0003 & 1.3402 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0003 & 1.2778 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 10} 
\label{tab:q3_2_10}
\end{table}
\subsection{Cluster 11 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0026 & 1.7024 \\ 
  2 & REACTOME\_GRB2\_EVENTS\_IN\_ERBB2\_SIGNALING & 0.0029 & 1.6670 \\ 
  3 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0023 & 1.6533 \\ 
  4 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0033 & 1.6504 \\ 
  5 & REACTOME\_SHC1\_EVENTS\_IN\_ERBB4\_SIGNALING & 0.0036 & 1.6413 \\ 
  6 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0034 & 1.6227 \\ 
  7 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0063 & 1.6013 \\ 
  8 & REACTOME\_PHASE\_4\_RESTING\_MEMBRANE\_POTENTIAL & 0.0049 & 1.5312 \\ 
  9 & REACTOME\_SHC1\_EVENTS\_IN\_ERBB2\_SIGNALING & 0.0045 & 1.5225 \\ 
  10 & REACTOME\_KERATINIZATION & 0.0051 & 1.4067 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 11} 
\label{tab:q3_2_11}
\end{table}
\subsection{Cluster 12 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_INWARDLY\_RECTIFYING\_K\_CHANNELS & 0.0129 & 1.6755 \\ 
  2 & REACTOME\_TRANSCRIPTIONAL\_REGULATION\_OF\_TESTIS\_DIFFERENTIATION & 0.0129 & 1.6597 \\ 
  3 & REACTOME\_THROMBIN\_SIGNALLING\_THROUGH\_PROTEINASE\_ACTIVATED\_RECEPTORS\_PARS & 0.0166 & 1.5895 \\ 
  4 & REACTOME\_G\_BETA\_GAMMA\_SIGNALLING\_THROUGH\_CDC42 & 0.0197 & 1.5515 \\ 
  5 & REACTOME\_G\_PROTEIN\_ACTIVATION & 0.0197 & 1.5349 \\ 
  6 & REACTOME\_PROSTACYCLIN\_SIGNALLING\_THROUGH\_PROSTACYCLIN\_RECEPTOR & 0.0197 & 1.5334 \\ 
  7 & REACTOME\_CA2\_PATHWAY & 0.0152 & 1.5332 \\ 
  8 & REACTOME\_PRESYNAPTIC\_FUNCTION\_OF\_KAINATE\_RECEPTORS & 0.0197 & 1.5142 \\ 
  9 & REACTOME\_PROTEIN\_UBIQUITINATION & 0.0191 & 1.4741 \\ 
  10 & REACTOME\_GABA\_RECEPTOR\_ACTIVATION & 0.0197 & 1.4606 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 12} 
\label{tab:q3_2_12}
\end{table}
\subsection{Cluster 13 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0042 & 1.6696 \\ 
  2 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0034 & 1.6272 \\ 
  3 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0060 & 1.5462 \\ 
  4 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0051 & 1.5298 \\ 
  5 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0051 & 1.5144 \\ 
  6 & REACTOME\_SIGNALING\_BY\_NTRK3\_TRKC & 0.0075 & 1.5009 \\ 
  7 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0053 & 1.4840 \\ 
  8 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0043 & 1.4785 \\ 
  9 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0055 & 1.4467 \\ 
  10 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0069 & 1.4063 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 13} 
\label{tab:q3_2_13}
\end{table}
\subsection{Cluster 14 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ABERRANT\_REGULATION\_OF\_MITOTIC\_G1\_S\_TRANSITION\_IN\_CANCER\_DUE\_TO\_RB1\_DEFECTS & 0.0021 & 2.0133 \\ 
  2 & REACTOME\_CTLA4\_INHIBITORY\_SIGNALING & 0.0021 & 1.9958 \\ 
  3 & REACTOME\_EPHA\_MEDIATED\_GROWTH\_CONE\_COLLAPSE & 0.0021 & 1.8312 \\ 
  4 & REACTOME\_ECM\_PROTEOGLYCANS & 0.0021 & 1.6973 \\ 
  5 & REACTOME\_APOPTOSIS & 0.0021 & 1.4012 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0001 & 1.3878 \\ 
  7 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0003 & 1.3816 \\ 
  8 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0021 & 1.2203 \\ 
  9 & REACTOME\_MEMBRANE\_TRAFFICKING & 0.0021 & 1.2163 \\ 
  10 & REACTOME\_CELL\_CYCLE & 0.0021 & 1.1944 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 14} 
\label{tab:q3_2_14}
\end{table}
\subsection{Cluster 15 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0015 & 1.7440 \\ 
  2 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0013 & 1.7018 \\ 
  3 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0020 & 1.6272 \\ 
  4 & REACTOME\_STING\_MEDIATED\_INDUCTION\_OF\_HOST\_IMMUNE\_RESPONSES & 0.0020 & 1.6188 \\ 
  5 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0039 & 1.6075 \\ 
  6 & REACTOME\_FCGR\_ACTIVATION & 0.0030 & 1.5710 \\ 
  7 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0028 & 1.5593 \\ 
  8 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0043 & 1.5579 \\ 
  9 & REACTOME\_TRAFFICKING\_OF\_GLUR2\_CONTAINING\_AMPA\_RECEPTORS & 0.0033 & 1.5535 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0031 & 1.5179 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 15} 
\label{tab:q3_2_15}
\end{table}
\subsection{Cluster 16 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0017 & 1.7510 \\ 
  2 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0012 & 1.6410 \\ 
  3 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0050 & 1.6382 \\ 
  4 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0056 & 1.5804 \\ 
  5 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0070 & 1.5606 \\ 
  6 & REACTOME\_DEFECTS\_OF\_CONTACT\_ACTIVATION\_SYSTEM\_CAS\_AND\_KALLIKREIN\_KININ\_SYSTEM\_KKS & 0.0065 & 1.5441 \\ 
  7 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0065 & 1.5197 \\ 
  8 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0065 & 1.4944 \\ 
  9 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0070 & 1.4715 \\ 
  10 & REACTOME\_NETRIN\_1\_SIGNALING & 0.0065 & 1.4354 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 16} 
\label{tab:q3_2_16}
\end{table}
\subsection{Cluster 17 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ABORTIVE\_ELONGATION\_OF\_HIV\_1\_TRANSCRIPT\_IN\_THE\_ABSENCE\_OF\_TAT & 0.0254 & 1.6010 \\ 
  2 & REACTOME\_CASPASE\_ACTIVATION\_VIA\_EXTRINSIC\_APOPTOTIC\_SIGNALLING\_PATHWAY & 0.0254 & 1.5182 \\ 
  3 & REACTOME\_BASIGIN\_INTERACTIONS & 0.0254 & 1.4889 \\ 
  4 & REACTOME\_B\_WICH\_COMPLEX\_POSITIVELY\_REGULATES\_RRNA\_EXPRESSION & 0.0254 & 1.4414 \\ 
  5 & REACTOME\_BASE\_EXCISION\_REPAIR & 0.0254 & 1.4301 \\ 
  6 & REACTOME\_CHOLESTEROL\_BIOSYNTHESIS & 0.0254 & 1.4174 \\ 
  7 & REACTOME\_CYCLIN\_A\_B1\_B2\_ASSOCIATED\_EVENTS\_DURING\_G2\_M\_TRANSITION & 0.0254 & 1.4031 \\ 
  8 & REACTOME\_CONSTITUTIVE\_SIGNALING\_BY\_ABERRANT\_PI3K\_IN\_CANCER & 0.0254 & 1.3634 \\ 
  9 & REACTOME\_CLASS\_I\_MHC\_MEDIATED\_ANTIGEN\_PROCESSING\_PRESENTATION & 0.0254 & 1.3000 \\ 
  10 & REACTOME\_ANTIGEN\_PROCESSING\_UBIQUITINATION\_PROTEASOME\_DEGRADATION & 0.0254 & 1.2755 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 17} 
\label{tab:q3_2_17}
\end{table}
\subsection{Cluster 18 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_CTLA4\_INHIBITORY\_SIGNALING & 0.0127 & 1.7256 \\ 
  2 & REACTOME\_TRANSCRIPTIONAL\_REGULATION\_BY\_RUNX1 & 0.0043 & 1.6641 \\ 
  3 & REACTOME\_DDX58\_IFIH1\_MEDIATED\_INDUCTION\_OF\_INTERFERON\_ALPHA\_BETA & 0.0127 & 1.6278 \\ 
  4 & REACTOME\_CHROMOSOME\_MAINTENANCE & 0.0127 & 1.6178 \\ 
  5 & REACTOME\_CARGO\_RECOGNITION\_FOR\_CLATHRIN\_MEDIATED\_ENDOCYTOSIS & 0.0127 & 1.5234 \\ 
  6 & REACTOME\_CYTOPROTECTION\_BY\_HMOX1 & 0.0127 & 1.4104 \\ 
  7 & REACTOME\_CLATHRIN\_MEDIATED\_ENDOCYTOSIS & 0.0127 & 1.3496 \\ 
  8 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0043 & 1.3221 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0127 & 1.2120 \\ 
  10 & REACTOME\_CELL\_CYCLE & 0.0127 & 1.1090 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 18} 
\label{tab:q3_2_18}
\end{table}
\subsection{Cluster 19 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0038 & 1.6617 \\ 
  2 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0045 & 1.6378 \\ 
  3 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0046 & 1.6245 \\ 
  4 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0034 & 1.6225 \\ 
  5 & REACTOME\_CARNITINE\_METABOLISM & 0.0069 & 1.6209 \\ 
  6 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0032 & 1.6086 \\ 
  7 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0033 & 1.6023 \\ 
  8 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0056 & 1.4972 \\ 
  9 & REACTOME\_FCGR\_ACTIVATION & 0.0070 & 1.4675 \\ 
  10 & REACTOME\_SIGNALING\_BY\_INSULIN\_RECEPTOR & 0.0068 & 1.4380 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 19} 
\label{tab:q3_2_19}
\end{table}
\subsection{Cluster 2 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0011 & 1.8467 \\ 
  2 & REACTOME\_PLATELET\_AGGREGATION\_PLUG\_FORMATION & 0.0050 & 1.5533 \\ 
  3 & REACTOME\_ACTIVATED\_NTRK2\_SIGNALS\_THROUGH\_FRS2\_AND\_FRS3 & 0.0072 & 1.5474 \\ 
  4 & REACTOME\_SIGNALING\_BY\_NTRK2\_TRKB & 0.0055 & 1.5408 \\ 
  5 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0053 & 1.5363 \\ 
  6 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0042 & 1.5143 \\ 
  7 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0058 & 1.5112 \\ 
  8 & REACTOME\_AUTOPHAGY & 0.0040 & 1.4072 \\ 
  9 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0058 & 1.4069 \\ 
  10 & REACTOME\_ARACHIDONIC\_ACID\_METABOLISM & 0.0079 & 1.3859 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 2} 
\label{tab:q3_2_2}
\end{table}
\subsection{Cluster 20 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ABERRANT\_REGULATION\_OF\_MITOTIC\_G1\_S\_TRANSITION\_IN\_CANCER\_DUE\_TO\_RB1\_DEFECTS & 0.0012 & 2.0793 \\ 
  2 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0006 & 2.0603 \\ 
  3 & REACTOME\_ESTROGEN\_DEPENDENT\_NUCLEAR\_EVENTS\_DOWNSTREAM\_OF\_ESR\_MEMBRANE\_SIGNALING & 0.0016 & 1.9204 \\ 
  4 & REACTOME\_ECM\_PROTEOGLYCANS & 0.0012 & 1.8194 \\ 
  5 & REACTOME\_SIGNALING\_BY\_WNT & 0.0012 & 1.4817 \\ 
  6 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0012 & 1.3437 \\ 
  7 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0012 & 1.3367 \\ 
  8 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0006 & 1.3004 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0014 & 1.2446 \\ 
  10 & REACTOME\_METABOLISM\_OF\_RNA & 0.0012 & 1.2423 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 20} 
\label{tab:q3_2_20}
\end{table}
\subsection{Cluster 21 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0024 & 1.7275 \\ 
  2 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0025 & 1.6662 \\ 
  3 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0040 & 1.5617 \\ 
  4 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0068 & 1.5558 \\ 
  5 & REACTOME\_ETHANOL\_OXIDATION & 0.0073 & 1.5066 \\ 
  6 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0068 & 1.5009 \\ 
  7 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0070 & 1.4759 \\ 
  8 & REACTOME\_AUTOPHAGY & 0.0021 & 1.4471 \\ 
  9 & REACTOME\_ANTIMICROBIAL\_PEPTIDES & 0.0074 & 1.4188 \\ 
  10 & REACTOME\_SELENOAMINO\_ACID\_METABOLISM & 0.0064 & 1.3840 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 21} 
\label{tab:q3_2_21}
\end{table}
\subsection{Cluster 22 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0015 & 1.6832 \\ 
  2 & REACTOME\_STING\_MEDIATED\_INDUCTION\_OF\_HOST\_IMMUNE\_RESPONSES & 0.0037 & 1.6185 \\ 
  3 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0030 & 1.6030 \\ 
  4 & REACTOME\_TRAFFICKING\_OF\_GLUR2\_CONTAINING\_AMPA\_RECEPTORS & 0.0032 & 1.5959 \\ 
  5 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0023 & 1.5711 \\ 
  6 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0026 & 1.5543 \\ 
  7 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0043 & 1.5531 \\ 
  8 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0047 & 1.5352 \\ 
  9 & REACTOME\_FCGR\_ACTIVATION & 0.0036 & 1.5198 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0039 & 1.4956 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 22} 
\label{tab:q3_2_22}
\end{table}
\subsection{Cluster 23 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0006 & 1.9712 \\ 
  2 & REACTOME\_ECM\_PROTEOGLYCANS & 0.0005 & 1.8322 \\ 
  3 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0003 & 1.4029 \\ 
  4 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0001 & 1.3976 \\ 
  5 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0005 & 1.3701 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0006 & 1.3460 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0003 & 1.3312 \\ 
  8 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0007 & 1.2941 \\ 
  9 & REACTOME\_TRANSPORT\_OF\_SMALL\_MOLECULES & 0.0006 & 1.2722 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0006 & 1.2230 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 23} 
\label{tab:q3_2_23}
\end{table}
\subsection{Cluster 24 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0033 & 1.6462 \\ 
  2 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0026 & 1.6370 \\ 
  3 & REACTOME\_NUCLEOTIDE\_LIKE\_PURINERGIC\_RECEPTORS & 0.0036 & 1.6230 \\ 
  4 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0048 & 1.5651 \\ 
  5 & REACTOME\_P2Y\_RECEPTORS & 0.0044 & 1.5589 \\ 
  6 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0034 & 1.5341 \\ 
  7 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0060 & 1.5010 \\ 
  8 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0048 & 1.4959 \\ 
  9 & REACTOME\_SIGNALING\_BY\_NTRK2\_TRKB & 0.0072 & 1.4718 \\ 
  10 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0083 & 1.4587 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 24} 
\label{tab:q3_2_24}
\end{table}
\subsection{Cluster 25 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0003 & 2.0347 \\ 
  2 & REACTOME\_HIV\_LIFE\_CYCLE & 0.0004 & 1.6196 \\ 
  3 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0001 & 1.4560 \\ 
  4 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0001 & 1.4367 \\ 
  5 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0000 & 1.4352 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0004 & 1.3587 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0001 & 1.3572 \\ 
  8 & REACTOME\_CELL\_CYCLE & 0.0001 & 1.3368 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0003 & 1.3185 \\ 
  10 & REACTOME\_INFECTIOUS\_DISEASE & 0.0004 & 1.2394 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 25} 
\label{tab:q3_2_25}
\end{table}
\subsection{Cluster 26 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0022 & 1.6643 \\ 
  2 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0024 & 1.6236 \\ 
  3 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0032 & 1.6087 \\ 
  4 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0024 & 1.5791 \\ 
  5 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0022 & 1.5567 \\ 
  6 & REACTOME\_ACTIVATED\_NTRK2\_SIGNALS\_THROUGH\_FRS2\_AND\_FRS3 & 0.0045 & 1.5338 \\ 
  7 & REACTOME\_BLOOD\_GROUP\_SYSTEMS\_BIOSYNTHESIS & 0.0067 & 1.5097 \\ 
  8 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0043 & 1.5082 \\ 
  9 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0046 & 1.4935 \\ 
  10 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0053 & 1.4543 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 26} 
\label{tab:q3_2_26}
\end{table}
\subsection{Cluster 27 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_RHOG\_GTPASE\_CYCLE & 0.0023 & 1.7602 \\ 
  2 & REACTOME\_INTERLEUKIN\_4\_AND\_INTERLEUKIN\_13\_SIGNALING & 0.0023 & 1.6243 \\ 
  3 & REACTOME\_RAC1\_GTPASE\_CYCLE & 0.0012 & 1.4778 \\ 
  4 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0002 & 1.4683 \\ 
  5 & REACTOME\_INTRACELLULAR\_SIGNALING\_BY\_SECOND\_MESSENGERS & 0.0012 & 1.4458 \\ 
  6 & REACTOME\_MAPK\_FAMILY\_SIGNALING\_CASCADES & 0.0012 & 1.4323 \\ 
  7 & REACTOME\_SIGNALING\_BY\_WNT & 0.0023 & 1.4249 \\ 
  8 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0012 & 1.3627 \\ 
  9 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0019 & 1.3406 \\ 
  10 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0012 & 1.3321 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 27} 
\label{tab:q3_2_27}
\end{table}
\subsection{Cluster 28 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0012 & 1.7632 \\ 
  2 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0024 & 1.7205 \\ 
  3 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0046 & 1.5977 \\ 
  4 & REACTOME\_NUCLEOTIDE\_LIKE\_PURINERGIC\_RECEPTORS & 0.0052 & 1.5898 \\ 
  5 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0050 & 1.5700 \\ 
  6 & REACTOME\_STING\_MEDIATED\_INDUCTION\_OF\_HOST\_IMMUNE\_RESPONSES & 0.0072 & 1.5350 \\ 
  7 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0043 & 1.5178 \\ 
  8 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0085 & 1.4063 \\ 
  9 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0075 & 1.4057 \\ 
  10 & REACTOME\_O\_GLYCOSYLATION\_OF\_TSR\_DOMAIN\_CONTAINING\_PROTEINS & 0.0084 & 1.4043 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 28} 
\label{tab:q3_2_28}
\end{table}
\subsection{Cluster 29 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_REGULATION\_OF\_GENE\_EXPRESSION\_IN\_LATE\_STAGE\_BRANCHING\_MORPHOGENESIS\_PANCREATIC\_BUD\_PRECURSOR\_CELLS & 0.0033 & 1.6437 \\ 
  2 & REACTOME\_RUNX3\_REGULATES\_NOTCH\_SIGNALING & 0.0048 & 1.5981 \\ 
  3 & REACTOME\_PURINE\_SALVAGE & 0.0044 & 1.5936 \\ 
  4 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0055 & 1.5717 \\ 
  5 & REACTOME\_CLASS\_C\_3\_METABOTROPIC\_GLUTAMATE\_PHEROMONE\_RECEPTORS & 0.0073 & 1.5289 \\ 
  6 & REACTOME\_ZBP1\_DAI\_MEDIATED\_INDUCTION\_OF\_TYPE\_I\_IFNS & 0.0080 & 1.5110 \\ 
  7 & REACTOME\_TAK1\_ACTIVATES\_NFKB\_BY\_PHOSPHORYLATION\_AND\_ACTIVATION\_OF\_IKKS\_COMPLEX & 0.0063 & 1.4865 \\ 
  8 & REACTOME\_ETHANOL\_OXIDATION & 0.0080 & 1.4822 \\ 
  9 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0073 & 1.4419 \\ 
  10 & REACTOME\_SIGNALING\_BY\_NOTCH3 & 0.0073 & 1.4345 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 29} 
\label{tab:q3_2_29}
\end{table}
\subsection{Cluster 3 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0009 & 1.8081 \\ 
  2 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0010 & 1.6894 \\ 
  3 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0020 & 1.6887 \\ 
  4 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0016 & 1.6031 \\ 
  5 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0035 & 1.5908 \\ 
  6 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0018 & 1.5747 \\ 
  7 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0032 & 1.5734 \\ 
  8 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0032 & 1.5712 \\ 
  9 & REACTOME\_TRAFFICKING\_OF\_GLUR2\_CONTAINING\_AMPA\_RECEPTORS & 0.0032 & 1.5699 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0030 & 1.5054 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 3} 
\label{tab:q3_2_3}
\end{table}
\subsection{Cluster 30 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_PURINE\_SALVAGE & 0.0072 & 1.4717 \\ 
  2 & REACTOME\_RESPONSE\_TO\_METAL\_IONS & 0.0087 & 1.4256 \\ 
  3 & REACTOME\_ERYTHROPOIETIN\_ACTIVATES\_RAS & 0.0088 & 1.4083 \\ 
  4 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0089 & 1.3923 \\ 
  5 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0088 & 1.3793 \\ 
  6 & REACTOME\_SUMOYLATION\_OF\_TRANSCRIPTION\_FACTORS & 0.0088 & 1.3762 \\ 
  7 & REACTOME\_P2Y\_RECEPTORS & 0.0107 & 1.3340 \\ 
  8 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0112 & 1.2782 \\ 
  9 & REACTOME\_POSITIVE\_EPIGENETIC\_REGULATION\_OF\_RRNA\_EXPRESSION & 0.0108 & 1.2545 \\ 
  10 & REACTOME\_SELENOAMINO\_ACID\_METABOLISM & 0.0096 & 1.2033 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 30} 
\label{tab:q3_2_30}
\end{table}
\subsection{Cluster 31 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0000 & 1.4988 \\ 
  2 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0001 & 1.4969 \\ 
  3 & REACTOME\_METABOLISM\_OF\_AMINO\_ACIDS\_AND\_DERIVATIVES & 0.0002 & 1.4960 \\ 
  4 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0002 & 1.4615 \\ 
  5 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0002 & 1.4296 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0002 & 1.4261 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0001 & 1.4229 \\ 
  8 & REACTOME\_CELLULAR\_RESPONSES\_TO\_STIMULI & 0.0001 & 1.3996 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0002 & 1.3986 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0000 & 1.3767 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 31} 
\label{tab:q3_2_31}
\end{table}
\subsection{Cluster 4 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0014 & 1.7325 \\ 
  2 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0006 & 1.7152 \\ 
  3 & REACTOME\_PURINE\_SALVAGE & 0.0030 & 1.6378 \\ 
  4 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0030 & 1.6186 \\ 
  5 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0045 & 1.5900 \\ 
  6 & REACTOME\_TRAF6\_MEDIATED\_IRF7\_ACTIVATION\_IN\_TLR7\_8\_OR\_9\_SIGNALING & 0.0039 & 1.5678 \\ 
  7 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0040 & 1.5433 \\ 
  8 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0033 & 1.4987 \\ 
  9 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0036 & 1.4816 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0051 & 1.3980 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 4} 
\label{tab:q3_2_4}
\end{table}
\subsection{Cluster 5 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0046 & 1.6412 \\ 
  2 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0047 & 1.6357 \\ 
  3 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0027 & 1.6337 \\ 
  4 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0051 & 1.6079 \\ 
  5 & REACTOME\_NUCLEOTIDE\_LIKE\_PURINERGIC\_RECEPTORS & 0.0052 & 1.5949 \\ 
  6 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0030 & 1.5936 \\ 
  7 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0061 & 1.5564 \\ 
  8 & REACTOME\_P2Y\_RECEPTORS & 0.0063 & 1.5502 \\ 
  9 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0048 & 1.5364 \\ 
  10 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0055 & 1.4801 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 5} 
\label{tab:q3_2_5}
\end{table}
\subsection{Cluster 6 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0001 & 2.0602 \\ 
  2 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0002 & 1.4485 \\ 
  3 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0002 & 1.4264 \\ 
  4 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0001 & 1.4182 \\ 
  5 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0003 & 1.4094 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0002 & 1.4088 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0001 & 1.4058 \\ 
  8 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0002 & 1.3673 \\ 
  9 & REACTOME\_TRANSPORT\_OF\_SMALL\_MOLECULES & 0.0002 & 1.3494 \\ 
  10 & REACTOME\_CELLULAR\_RESPONSES\_TO\_STIMULI & 0.0003 & 1.3099 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 6} 
\label{tab:q3_2_6}
\end{table}
\subsection{Cluster 7 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_CA2\_PATHWAY & 0.0165 & 1.6989 \\ 
  2 & REACTOME\_G\_PROTEIN\_BETA\_GAMMA\_SIGNALLING & 0.0165 & 1.6980 \\ 
  3 & REACTOME\_GLUCAGON\_TYPE\_LIGAND\_RECEPTORS & 0.0165 & 1.6925 \\ 
  4 & REACTOME\_ACTIVATION\_OF\_KAINATE\_RECEPTORS\_UPON\_GLUTAMATE\_BINDING & 0.0165 & 1.6829 \\ 
  5 & REACTOME\_G\_PROTEIN\_ACTIVATION & 0.0165 & 1.6700 \\ 
  6 & REACTOME\_GRB2\_EVENTS\_IN\_ERBB2\_SIGNALING & 0.0165 & 1.6402 \\ 
  7 & REACTOME\_G2\_M\_DNA\_DAMAGE\_CHECKPOINT & 0.0165 & 1.6338 \\ 
  8 & REACTOME\_EXTRA\_NUCLEAR\_ESTROGEN\_SIGNALING & 0.0165 & 1.6292 \\ 
  9 & REACTOME\_G\_ALPHA\_Q\_SIGNALLING\_EVENTS & 0.0165 & 1.5390 \\ 
  10 & REACTOME\_GPCR\_LIGAND\_BINDING & 0.0165 & 1.3763 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 7} 
\label{tab:q3_2_7}
\end{table}
\subsection{Cluster 8 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0111 & 2.1276 \\ 
  2 & REACTOME\_BUTYROPHILIN\_BTN\_FAMILY\_INTERACTIONS & 0.0224 & 1.9228 \\ 
  3 & REACTOME\_INTERLEUKIN\_2\_SIGNALING & 0.0224 & 1.8989 \\ 
  4 & REACTOME\_CTLA4\_INHIBITORY\_SIGNALING & 0.0224 & 1.8529 \\ 
  5 & REACTOME\_ABERRANT\_REGULATION\_OF\_MITOTIC\_G1\_S\_TRANSITION\_IN\_CANCER\_DUE\_TO\_RB1\_DEFECTS & 0.0224 & 1.7735 \\ 
  6 & REACTOME\_CD209\_DC\_SIGN\_SIGNALING & 0.0224 & 1.6381 \\ 
  7 & REACTOME\_EPHA\_MEDIATED\_GROWTH\_CONE\_COLLAPSE & 0.0224 & 1.5482 \\ 
  8 & REACTOME\_ECM\_PROTEOGLYCANS & 0.0224 & 1.3735 \\ 
  9 & REACTOME\_DDX58\_IFIH1\_MEDIATED\_INDUCTION\_OF\_INTERFERON\_ALPHA\_BETA & 0.0224 & 1.3536 \\ 
  10 & REACTOME\_GENE\_SILENCING\_BY\_RNA & 0.0224 & 1.3474 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 8} 
\label{tab:q3_2_8}
\end{table}
\subsection{Cluster 9 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0017 & 1.7368 \\ 
  2 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0028 & 1.6908 \\ 
  3 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0008 & 1.6890 \\ 
  4 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0017 & 1.6064 \\ 
  5 & REACTOME\_ACTIVATED\_NTRK2\_SIGNALS\_THROUGH\_FRS2\_AND\_FRS3 & 0.0050 & 1.6000 \\ 
  6 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0056 & 1.5535 \\ 
  7 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0031 & 1.5161 \\ 
  8 & REACTOME\_FCGR\_ACTIVATION & 0.0042 & 1.4837 \\ 
  9 & REACTOME\_CD22\_MEDIATED\_BCR\_REGULATION & 0.0064 & 1.4756 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0042 & 1.4618 \\ 
   \hline
\end{tabularx}
\caption{Top 10 up-regulated pathways for cluster 9} 
\label{tab:q3_2_9}
\end{table}
\section{Q3.2 Top 10 up-regulated pathways with adjustement for confondant effect}
\subsection{Cluster 0 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0014 & 1.8397 \\ 
  2 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0058 & 1.7446 \\ 
  3 & REACTOME\_CYTOSOLIC\_SULFONATION\_OF\_SMALL\_MOLECULES & 0.0062 & 1.6324 \\ 
  4 & REACTOME\_PURINE\_SALVAGE & 0.0067 & 1.6313 \\ 
  5 & REACTOME\_PYRIMIDINE\_SALVAGE & 0.0077 & 1.5936 \\ 
  6 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0081 & 1.5600 \\ 
  7 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0090 & 1.5038 \\ 
  8 & REACTOME\_HYALURONAN\_METABOLISM & 0.0086 & 1.5005 \\ 
  9 & REACTOME\_O\_GLYCOSYLATION\_OF\_TSR\_DOMAIN\_CONTAINING\_PROTEINS & 0.0098 & 1.4624 \\ 
  10 & REACTOME\_METABOLISM\_OF\_NUCLEOTIDES & 0.0101 & 1.3500 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 0 with adjustement for confondant effect} 
\label{tab:q3_2_conf_0}
\end{table}
\subsection{Cluster 1 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0191 & 1.7800 \\ 
  2 & REACTOME\_INFLAMMASOMES & 0.0191 & 1.6472 \\ 
  3 & REACTOME\_ADHERENS\_JUNCTIONS\_INTERACTIONS & 0.0191 & 1.6328 \\ 
  4 & REACTOME\_ACTIVATION\_OF\_THE\_PRE\_REPLICATIVE\_COMPLEX & 0.0191 & 1.5673 \\ 
  5 & REACTOME\_TP53\_REGULATES\_TRANSCRIPTION\_OF\_SEVERAL\_ADDITIONAL\_CELL\_DEATH\_GENES\_WHOSE\_SPECIFIC\_ROLES\_IN\_P53\_DEPENDENT\_APOPTOSIS\_REMAIN\_UNCERTAIN & 0.0191 & 1.5603 \\ 
  6 & REACTOME\_RECYCLING\_OF\_BILE\_ACIDS\_AND\_SALTS & 0.0191 & 1.5589 \\ 
  7 & REACTOME\_BBSOME\_MEDIATED\_CARGO\_TARGETING\_TO\_CILIUM & 0.0194 & 1.5548 \\ 
  8 & REACTOME\_PROLONGED\_ERK\_ACTIVATION\_EVENTS & 0.0194 & 1.5480 \\ 
  9 & REACTOME\_PURINERGIC\_SIGNALING\_IN\_LEISHMANIASIS\_INFECTION & 0.0191 & 1.5433 \\ 
  10 & REACTOME\_SWITCHING\_OF\_ORIGINS\_TO\_A\_POST\_REPLICATIVE\_STATE & 0.0191 & 1.5049 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 1 with adjustement for confondant effect} 
\label{tab:q3_2_conf_1}
\end{table}
\subsection{Cluster 2 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0080 & 1.7707 \\ 
  2 & REACTOME\_ACTIVATED\_NTRK2\_SIGNALS\_THROUGH\_FRS2\_AND\_FRS3 & 0.0109 & 1.6270 \\ 
  3 & REACTOME\_REGULATION\_OF\_TP53\_ACTIVITY\_THROUGH\_ACETYLATION & 0.0110 & 1.5969 \\ 
  4 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0127 & 1.5906 \\ 
  5 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0127 & 1.5471 \\ 
  6 & REACTOME\_CYTOSOLIC\_SULFONATION\_OF\_SMALL\_MOLECULES & 0.0112 & 1.5414 \\ 
  7 & REACTOME\_FOXO\_MEDIATED\_TRANSCRIPTION\_OF\_CELL\_CYCLE\_GENES & 0.0146 & 1.5037 \\ 
  8 & REACTOME\_SEALING\_OF\_THE\_NUCLEAR\_ENVELOPE\_NE\_BY\_ESCRT\_III & 0.0140 & 1.4774 \\ 
  9 & REACTOME\_ARACHIDONIC\_ACID\_METABOLISM & 0.0140 & 1.3798 \\ 
  10 & REACTOME\_AUTOPHAGY & 0.0112 & 1.3412 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 2 with adjustement for confondant effect} 
\label{tab:q3_2_conf_2}
\end{table}
\subsection{Cluster 3 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_STING\_MEDIATED\_INDUCTION\_OF\_HOST\_IMMUNE\_RESPONSES & 0.0013 & 1.8318 \\ 
  2 & REACTOME\_IRF3\_MEDIATED\_INDUCTION\_OF\_TYPE\_I\_IFN & 0.0014 & 1.7682 \\ 
  3 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0074 & 1.5476 \\ 
  4 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0040 & 1.5420 \\ 
  5 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0085 & 1.5411 \\ 
  6 & REACTOME\_RESPONSE\_TO\_METAL\_IONS & 0.0094 & 1.5201 \\ 
  7 & REACTOME\_ZINC\_TRANSPORTERS & 0.0075 & 1.5138 \\ 
  8 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0069 & 1.5051 \\ 
  9 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0094 & 1.4967 \\ 
  10 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0080 & 1.4778 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 3 with adjustement for confondant effect} 
\label{tab:q3_2_conf_3}
\end{table}
\subsection{Cluster 4 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_PURINE\_SALVAGE & 0.0007 & 1.8566 \\ 
  2 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0023 & 1.7881 \\ 
  3 & REACTOME\_TRANSPORT\_OF\_CONNEXONS\_TO\_THE\_PLASMA\_MEMBRANE & 0.0062 & 1.6337 \\ 
  4 & REACTOME\_SYNTHESIS\_OF\_PIPS\_AT\_THE\_EARLY\_ENDOSOME\_MEMBRANE & 0.0072 & 1.5874 \\ 
  5 & REACTOME\_PHYSIOLOGICAL\_FACTORS & 0.0103 & 1.5247 \\ 
  6 & REACTOME\_POST\_CHAPERONIN\_TUBULIN\_FOLDING\_PATHWAY & 0.0122 & 1.4944 \\ 
  7 & REACTOME\_GAP\_JUNCTION\_ASSEMBLY & 0.0110 & 1.4900 \\ 
  8 & REACTOME\_ACTIVATION\_OF\_AMPK\_DOWNSTREAM\_OF\_NMDARS & 0.0117 & 1.4782 \\ 
  9 & REACTOME\_SEALING\_OF\_THE\_NUCLEAR\_ENVELOPE\_NE\_BY\_ESCRT\_III & 0.0120 & 1.4659 \\ 
  10 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0117 & 1.4539 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 4 with adjustement for confondant effect} 
\label{tab:q3_2_conf_4}
\end{table}
\subsection{Cluster 5 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0029 & 1.7068 \\ 
  2 & REACTOME\_RAF\_INDEPENDENT\_MAPK1\_3\_ACTIVATION & 0.0042 & 1.7046 \\ 
  3 & REACTOME\_SIGNALING\_BY\_NTRK3\_TRKC & 0.0046 & 1.6581 \\ 
  4 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0054 & 1.5881 \\ 
  5 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0069 & 1.5578 \\ 
  6 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0107 & 1.5550 \\ 
  7 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0107 & 1.5467 \\ 
  8 & REACTOME\_SIGNALING\_BY\_INSULIN\_RECEPTOR & 0.0058 & 1.5450 \\ 
  9 & REACTOME\_GASTRIN\_CREB\_SIGNALLING\_PATHWAY\_VIA\_PKC\_AND\_MAPK & 0.0107 & 1.5148 \\ 
  10 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0109 & 1.4599 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 5 with adjustement for confondant effect} 
\label{tab:q3_2_conf_5}
\end{table}
\subsection{Cluster 6 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_TRANSCRIPTIONAL\_REGULATION\_BY\_RUNX1 & 0.0001 & 1.7394 \\ 
  2 & REACTOME\_SIGNALING\_BY\_WNT & 0.0001 & 1.6567 \\ 
  3 & REACTOME\_SIGNALING\_BY\_NUCLEAR\_RECEPTORS & 0.0002 & 1.5887 \\ 
  4 & REACTOME\_METABOLISM\_OF\_RNA & 0.0000 & 1.5194 \\ 
  5 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0000 & 1.5005 \\ 
  6 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0001 & 1.4847 \\ 
  7 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0001 & 1.4823 \\ 
  8 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0001 & 1.4335 \\ 
  9 & REACTOME\_TRANSPORT\_OF\_SMALL\_MOLECULES & 0.0001 & 1.4275 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0001 & 1.3371 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 6 with adjustement for confondant effect} 
\label{tab:q3_2_conf_6}
\end{table}
\subsection{Cluster 7 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_COOPERATION\_OF\_PDCL\_PHLP1\_AND\_TRIC\_CCT\_IN\_G\_PROTEIN\_BETA\_FOLDING & 0.0175 & 1.6514 \\ 
  2 & REACTOME\_ACTIVATION\_OF\_KAINATE\_RECEPTORS\_UPON\_GLUTAMATE\_BINDING & 0.0175 & 1.6432 \\ 
  3 & REACTOME\_ADP\_SIGNALLING\_THROUGH\_P2Y\_PURINOCEPTOR\_1 & 0.0175 & 1.6265 \\ 
  4 & REACTOME\_ADRENALINE\_NORADRENALINE\_INHIBITS\_INSULIN\_SECRETION & 0.0175 & 1.6238 \\ 
  5 & REACTOME\_ACTIVATION\_OF\_ATR\_IN\_RESPONSE\_TO\_REPLICATION\_STRESS & 0.0175 & 1.6201 \\ 
  6 & REACTOME\_CA2\_PATHWAY & 0.0175 & 1.6194 \\ 
  7 & REACTOME\_EXTRA\_NUCLEAR\_ESTROGEN\_SIGNALING & 0.0175 & 1.5929 \\ 
  8 & REACTOME\_CONSTITUTIVE\_SIGNALING\_BY\_EGFRVIII & 0.0175 & 1.5801 \\ 
  9 & REACTOME\_DEUBIQUITINATION & 0.0175 & 1.4003 \\ 
  10 & REACTOME\_CLASS\_A\_1\_RHODOPSIN\_LIKE\_RECEPTORS & 0.0175 & 1.3639 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 7 with adjustement for confondant effect} 
\label{tab:q3_2_conf_7}
\end{table}
\subsection{Cluster 8 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_NEGATIVE\_REGULATION\_OF\_NMDA\_RECEPTOR\_MEDIATED\_NEURONAL\_TRANSMISSION & 0.0017 & 2.0733 \\ 
  2 & REACTOME\_TP53\_REGULATES\_TRANSCRIPTION\_OF\_SEVERAL\_ADDITIONAL\_CELL\_DEATH\_GENES\_WHOSE\_SPECIFIC\_ROLES\_IN\_P53\_DEPENDENT\_APOPTOSIS\_REMAIN\_UNCERTAIN & 0.0027 & 1.9570 \\ 
  3 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0037 & 1.9018 \\ 
  4 & REACTOME\_INTRACELLULAR\_SIGNALING\_BY\_SECOND\_MESSENGERS & 0.0017 & 1.3656 \\ 
  5 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0017 & 1.3532 \\ 
  6 & REACTOME\_MAPK\_FAMILY\_SIGNALING\_CASCADES & 0.0018 & 1.3526 \\ 
  7 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0017 & 1.2894 \\ 
  8 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0017 & 1.2883 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0017 & 1.2836 \\ 
  10 & REACTOME\_TRANSPORT\_OF\_SMALL\_MOLECULES & 0.0017 & 1.2111 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 8 with adjustement for confondant effect} 
\label{tab:q3_2_conf_8}
\end{table}
\subsection{Cluster 9 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0037 & 1.6607 \\ 
  2 & REACTOME\_GABA\_SYNTHESIS\_RELEASE\_REUPTAKE\_AND\_DEGRADATION & 0.0064 & 1.6339 \\ 
  3 & REACTOME\_O\_GLYCOSYLATION\_OF\_TSR\_DOMAIN\_CONTAINING\_PROTEINS & 0.0050 & 1.6308 \\ 
  4 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0118 & 1.5904 \\ 
  5 & REACTOME\_ADHERENS\_JUNCTIONS\_INTERACTIONS & 0.0087 & 1.5822 \\ 
  6 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0062 & 1.5636 \\ 
  7 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0055 & 1.5632 \\ 
  8 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0064 & 1.5590 \\ 
  9 & REACTOME\_FCGR\_ACTIVATION & 0.0072 & 1.5142 \\ 
  10 & REACTOME\_CD22\_MEDIATED\_BCR\_REGULATION & 0.0119 & 1.4662 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 9 with adjustement for confondant effect} 
\label{tab:q3_2_conf_9}
\end{table}
\subsection{Cluster 10 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ESR\_MEDIATED\_SIGNALING & 0.0001 & 1.7877 \\ 
  2 & REACTOME\_SIGNALING\_BY\_NUCLEAR\_RECEPTORS & 0.0001 & 1.6804 \\ 
  3 & REACTOME\_SIGNALING\_BY\_RECEPTOR\_TYROSINE\_KINASES & 0.0001 & 1.4854 \\ 
  4 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0001 & 1.4829 \\ 
  5 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0001 & 1.4740 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0000 & 1.4695 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0000 & 1.4688 \\ 
  8 & REACTOME\_MEMBRANE\_TRAFFICKING & 0.0001 & 1.3905 \\ 
  9 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0001 & 1.3871 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0001 & 1.3392 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 10 with adjustement for confondant effect} 
\label{tab:q3_2_conf_10}
\end{table}
\subsection{Cluster 11 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_CYTOSOLIC\_SULFONATION\_OF\_SMALL\_MOLECULES & 0.0088 & 1.8447 \\ 
  2 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0138 & 1.6831 \\ 
  3 & REACTOME\_INSULIN\_RECEPTOR\_RECYCLING & 0.0131 & 1.6819 \\ 
  4 & REACTOME\_SIGNALING\_BY\_ACTIVIN & 0.0147 & 1.6507 \\ 
  5 & REACTOME\_BASIGIN\_INTERACTIONS & 0.0138 & 1.5869 \\ 
  6 & REACTOME\_RESOLUTION\_OF\_D\_LOOP\_STRUCTURES & 0.0138 & 1.5776 \\ 
  7 & REACTOME\_TRANSFERRIN\_ENDOCYTOSIS\_AND\_RECYCLING & 0.0147 & 1.5703 \\ 
  8 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0148 & 1.5591 \\ 
  9 & REACTOME\_SIGNALING\_BY\_INSULIN\_RECEPTOR & 0.0158 & 1.4907 \\ 
  10 & REACTOME\_POTASSIUM\_CHANNELS & 0.0147 & 1.4025 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 11 with adjustement for confondant effect} 
\label{tab:q3_2_conf_11}
\end{table}
\subsection{Cluster 12 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_TRANSCRIPTIONAL\_REGULATION\_OF\_TESTIS\_DIFFERENTIATION & 0.0126 & 1.6713 \\ 
  2 & REACTOME\_HDMS\_DEMETHYLATE\_HISTONES & 0.0126 & 1.6630 \\ 
  3 & REACTOME\_INWARDLY\_RECTIFYING\_K\_CHANNELS & 0.0126 & 1.6524 \\ 
  4 & REACTOME\_NEGATIVE\_REGULATORS\_OF\_DDX58\_IFIH1\_SIGNALING & 0.0126 & 1.6252 \\ 
  5 & REACTOME\_THROMBIN\_SIGNALLING\_THROUGH\_PROTEINASE\_ACTIVATED\_RECEPTORS\_PARS & 0.0132 & 1.5931 \\ 
  6 & REACTOME\_G\_BETA\_GAMMA\_SIGNALLING\_THROUGH\_CDC42 & 0.0166 & 1.5856 \\ 
  7 & REACTOME\_BLOOD\_GROUP\_SYSTEMS\_BIOSYNTHESIS & 0.0166 & 1.5781 \\ 
  8 & REACTOME\_PRESYNAPTIC\_FUNCTION\_OF\_KAINATE\_RECEPTORS & 0.0211 & 1.5491 \\ 
  9 & REACTOME\_KERATAN\_SULFATE\_KERATIN\_METABOLISM & 0.0211 & 1.5059 \\ 
  10 & REACTOME\_CA2\_PATHWAY & 0.0207 & 1.4754 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 12 with adjustement for confondant effect} 
\label{tab:q3_2_conf_12}
\end{table}
\subsection{Cluster 13 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_BIOTIN\_TRANSPORT\_AND\_METABOLISM & 0.0091 & 1.6129 \\ 
  2 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0124 & 1.5218 \\ 
  3 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0088 & 1.5119 \\ 
  4 & REACTOME\_INSULIN\_RECEPTOR\_RECYCLING & 0.0099 & 1.4999 \\ 
  5 & REACTOME\_SUMOYLATION\_OF\_IMMUNE\_RESPONSE\_PROTEINS & 0.0138 & 1.4929 \\ 
  6 & REACTOME\_NCAM1\_INTERACTIONS & 0.0092 & 1.4859 \\ 
  7 & REACTOME\_ADHERENS\_JUNCTIONS\_INTERACTIONS & 0.0144 & 1.4766 \\ 
  8 & REACTOME\_RHOV\_GTPASE\_CYCLE & 0.0165 & 1.4275 \\ 
  9 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0155 & 1.3808 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0135 & 1.3637 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 13 with adjustement for confondant effect} 
\label{tab:q3_2_conf_13}
\end{table}
\subsection{Cluster 14 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_DEADENYLATION\_OF\_MRNA & 0.0067 & 1.8682 \\ 
  2 & REACTOME\_NUCLEAR\_RECEPTOR\_TRANSCRIPTION\_PATHWAY & 0.0077 & 1.7228 \\ 
  3 & REACTOME\_RHOC\_GTPASE\_CYCLE & 0.0083 & 1.6190 \\ 
  4 & REACTOME\_CILIUM\_ASSEMBLY & 0.0067 & 1.4961 \\ 
  5 & REACTOME\_SIGNALING\_BY\_NUCLEAR\_RECEPTORS & 0.0067 & 1.4713 \\ 
  6 & REACTOME\_ORGANELLE\_BIOGENESIS\_AND\_MAINTENANCE & 0.0077 & 1.4116 \\ 
  7 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0048 & 1.3712 \\ 
  8 & REACTOME\_METABOLISM\_OF\_AMINO\_ACIDS\_AND\_DERIVATIVES & 0.0077 & 1.3671 \\ 
  9 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0048 & 1.3035 \\ 
  10 & REACTOME\_DEVELOPMENTAL\_BIOLOGY & 0.0083 & 1.1967 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 14 with adjustement for confondant effect} 
\label{tab:q3_2_conf_14}
\end{table}
\subsection{Cluster 15 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0055 & 1.5843 \\ 
  2 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0045 & 1.5842 \\ 
  3 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0056 & 1.5832 \\ 
  4 & REACTOME\_IRF3\_MEDIATED\_INDUCTION\_OF\_TYPE\_I\_IFN & 0.0064 & 1.5585 \\ 
  5 & REACTOME\_RAF\_INDEPENDENT\_MAPK1\_3\_ACTIVATION & 0.0064 & 1.5351 \\ 
  6 & REACTOME\_ADHERENS\_JUNCTIONS\_INTERACTIONS & 0.0069 & 1.5304 \\ 
  7 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0070 & 1.4938 \\ 
  8 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0056 & 1.4902 \\ 
  9 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0072 & 1.4591 \\ 
  10 & REACTOME\_CD22\_MEDIATED\_BCR\_REGULATION & 0.0074 & 1.4377 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 15 with adjustement for confondant effect} 
\label{tab:q3_2_conf_15}
\end{table}
\subsection{Cluster 16 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0016 & 1.6775 \\ 
  2 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0048 & 1.5823 \\ 
  3 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0048 & 1.5642 \\ 
  4 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0049 & 1.5624 \\ 
  5 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0074 & 1.5280 \\ 
  6 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0062 & 1.5196 \\ 
  7 & REACTOME\_ENDOSOMAL\_VACUOLAR\_PATHWAY & 0.0066 & 1.5173 \\ 
  8 & REACTOME\_DCC\_MEDIATED\_ATTRACTIVE\_SIGNALING & 0.0075 & 1.5133 \\ 
  9 & REACTOME\_AMINE\_LIGAND\_BINDING\_RECEPTORS & 0.0094 & 1.4672 \\ 
  10 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0052 & 1.4656 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 16 with adjustement for confondant effect} 
\label{tab:q3_2_conf_16}
\end{table}
\subsection{Cluster 17 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_G1\_S\_SPECIFIC\_TRANSCRIPTION & 0.0111 & 1.8989 \\ 
  2 & REACTOME\_INTERCONVERSION\_OF\_NUCLEOTIDE\_DI\_AND\_TRIPHOSPHATES & 0.0111 & 1.8786 \\ 
  3 & REACTOME\_APOPTOTIC\_CLEAVAGE\_OF\_CELL\_ADHESION\_PROTEINS & 0.0249 & 1.5718 \\ 
  4 & REACTOME\_BASE\_EXCISION\_REPAIR & 0.0249 & 1.5286 \\ 
  5 & REACTOME\_B\_WICH\_COMPLEX\_POSITIVELY\_REGULATES\_RRNA\_EXPRESSION & 0.0249 & 1.5117 \\ 
  6 & REACTOME\_ACTIVATION\_OF\_GENE\_EXPRESSION\_BY\_SREBF\_SREBP & 0.0249 & 1.4265 \\ 
  7 & REACTOME\_BIOLOGICAL\_OXIDATIONS & 0.0249 & 1.4180 \\ 
  8 & REACTOME\_ABORTIVE\_ELONGATION\_OF\_HIV\_1\_TRANSCRIPT\_IN\_THE\_ABSENCE\_OF\_TAT & 0.0249 & 1.4167 \\ 
  9 & REACTOME\_BASIGIN\_INTERACTIONS & 0.0249 & 1.4132 \\ 
  10 & REACTOME\_ANTIGEN\_PROCESSING\_UBIQUITINATION\_PROTEASOME\_DEGRADATION & 0.0249 & 1.2880 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 17 with adjustement for confondant effect} 
\label{tab:q3_2_conf_17}
\end{table}
\subsection{Cluster 18 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_MET\_RECEPTOR\_RECYCLING & 0.0188 & 2.7131 \\ 
  2 & REACTOME\_LOSS\_OF\_FUNCTION\_OF\_MECP2\_IN\_RETT\_SYNDROME & 0.0173 & 2.5978 \\ 
  3 & REACTOME\_CASPASE\_ACTIVATION\_VIA\_DEATH\_RECEPTORS\_IN\_THE\_PRESENCE\_OF\_LIGAND & 0.0192 & 2.4909 \\ 
  4 & REACTOME\_TRAF6\_MEDIATED\_INDUCTION\_OF\_TAK1\_COMPLEX\_WITHIN\_TLR4\_COMPLEX & 0.0192 & 2.4909 \\ 
  5 & REACTOME\_MECP2\_REGULATES\_NEURONAL\_RECEPTORS\_AND\_CHANNELS & 0.0152 & 2.4373 \\ 
  6 & REACTOME\_ACTIVATION\_OF\_IRF3\_IRF7\_MEDIATED\_BY\_TBK1\_IKK\_EPSILON & 0.0194 & 2.4362 \\ 
  7 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0033 & 1.4240 \\ 
  8 & REACTOME\_NEUTROPHIL\_DEGRANULATION & 0.0152 & 1.3372 \\ 
  9 & REACTOME\_CYTOPROTECTION\_BY\_HMOX1 & 0.0173 & 1.2471 \\ 
  10 & REACTOME\_CELLULAR\_RESPONSE\_TO\_CHEMICAL\_STRESS & 0.0173 & 1.2470 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 18 with adjustement for confondant effect} 
\label{tab:q3_2_conf_18}
\end{table}
\subsection{Cluster 19 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_INSULIN\_RECEPTOR\_RECYCLING & 0.0162 & 1.8236 \\ 
  2 & REACTOME\_GABA\_SYNTHESIS\_RELEASE\_REUPTAKE\_AND\_DEGRADATION & 0.0162 & 1.6381 \\ 
  3 & REACTOME\_ENERGY\_DEPENDENT\_REGULATION\_OF\_MTOR\_BY\_LKB1\_AMPK & 0.0162 & 1.6317 \\ 
  4 & REACTOME\_ACYL\_CHAIN\_REMODELLING\_OF\_PI & 0.0162 & 1.6181 \\ 
  5 & REACTOME\_FORMATION\_OF\_FIBRIN\_CLOT\_CLOTTING\_CASCADE & 0.0162 & 1.6156 \\ 
  6 & REACTOME\_DEFECTS\_OF\_CONTACT\_ACTIVATION\_SYSTEM\_CAS\_AND\_KALLIKREIN\_KININ\_SYSTEM\_KKS & 0.0162 & 1.6072 \\ 
  7 & REACTOME\_BLOOD\_GROUP\_SYSTEMS\_BIOSYNTHESIS & 0.0162 & 1.6059 \\ 
  8 & REACTOME\_CREB1\_PHOSPHORYLATION\_THROUGH\_NMDA\_RECEPTOR\_MEDIATED\_ACTIVATION\_OF\_RAS\_SIGNALING & 0.0162 & 1.6050 \\ 
  9 & REACTOME\_AMINO\_ACIDS\_REGULATE\_MTORC1 & 0.0162 & 1.5214 \\ 
  10 & REACTOME\_ACTIVATION\_OF\_NMDA\_RECEPTORS\_AND\_POSTSYNAPTIC\_EVENTS & 0.0162 & 1.5165 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 19 with adjustement for confondant effect} 
\label{tab:q3_2_conf_19}
\end{table}
\subsection{Cluster 20 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ABERRANT\_REGULATION\_OF\_MITOTIC\_G1\_S\_TRANSITION\_IN\_CANCER\_DUE\_TO\_RB1\_DEFECTS & 0.0029 & 1.9536 \\ 
  2 & REACTOME\_RUNX2\_REGULATES\_BONE\_DEVELOPMENT & 0.0029 & 1.9502 \\ 
  3 & REACTOME\_RUNX2\_REGULATES\_OSTEOBLAST\_DIFFERENTIATION & 0.0088 & 1.9044 \\ 
  4 & REACTOME\_FORMATION\_OF\_SENESCENCE\_ASSOCIATED\_HETEROCHROMATIN\_FOCI\_SAHF & 0.0106 & 1.8948 \\ 
  5 & REACTOME\_REGULATION\_OF\_RUNX1\_EXPRESSION\_AND\_ACTIVITY & 0.0106 & 1.6825 \\ 
  6 & REACTOME\_DDX58\_IFIH1\_MEDIATED\_INDUCTION\_OF\_INTERFERON\_ALPHA\_BETA & 0.0106 & 1.5555 \\ 
  7 & REACTOME\_SIGNALING\_BY\_WNT & 0.0029 & 1.4974 \\ 
  8 & REACTOME\_APOPTOSIS & 0.0106 & 1.4549 \\ 
  9 & REACTOME\_PROGRAMMED\_CELL\_DEATH & 0.0106 & 1.3193 \\ 
  10 & REACTOME\_EXTRACELLULAR\_MATRIX\_ORGANIZATION & 0.0106 & 1.3028 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 20 with adjustement for confondant effect} 
\label{tab:q3_2_conf_20}
\end{table}
\subsection{Cluster 21 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_ACTIVATION\_OF\_RAC1 & 0.0031 & 1.6999 \\ 
  2 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0048 & 1.6851 \\ 
  3 & REACTOME\_SYNTHESIS\_SECRETION\_AND\_DEACYLATION\_OF\_GHRELIN & 0.0069 & 1.5864 \\ 
  4 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0039 & 1.5581 \\ 
  5 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0059 & 1.5486 \\ 
  6 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0051 & 1.5481 \\ 
  7 & REACTOME\_PURINE\_SALVAGE & 0.0068 & 1.5461 \\ 
  8 & REACTOME\_ANTIMICROBIAL\_PEPTIDES & 0.0040 & 1.5102 \\ 
  9 & REACTOME\_TRAF6\_MEDIATED\_IRF7\_ACTIVATION\_IN\_TLR7\_8\_OR\_9\_SIGNALING & 0.0083 & 1.4830 \\ 
  10 & REACTOME\_SELENOAMINO\_ACID\_METABOLISM & 0.0078 & 1.3581 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 21 with adjustement for confondant effect} 
\label{tab:q3_2_conf_21}
\end{table}
\subsection{Cluster 22 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0018 & 1.6467 \\ 
  2 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0037 & 1.5756 \\ 
  3 & REACTOME\_SIGNALING\_BY\_NTRK3\_TRKC & 0.0063 & 1.5196 \\ 
  4 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0044 & 1.4994 \\ 
  5 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0049 & 1.4725 \\ 
  6 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0052 & 1.4561 \\ 
  7 & REACTOME\_ADHERENS\_JUNCTIONS\_INTERACTIONS & 0.0083 & 1.4080 \\ 
  8 & REACTOME\_FCGR\_ACTIVATION & 0.0069 & 1.3912 \\ 
  9 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0077 & 1.3813 \\ 
  10 & REACTOME\_CD22\_MEDIATED\_BCR\_REGULATION & 0.0083 & 1.3598 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 22 with adjustement for confondant effect} 
\label{tab:q3_2_conf_22}
\end{table}
\subsection{Cluster 23 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_BRANCHED\_CHAIN\_AMINO\_ACID\_CATABOLISM & 0.0004 & 1.9736 \\ 
  2 & REACTOME\_METABOLISM\_OF\_AMINO\_ACIDS\_AND\_DERIVATIVES & 0.0002 & 1.5545 \\ 
  3 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0002 & 1.4726 \\ 
  4 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0001 & 1.4627 \\ 
  5 & REACTOME\_METABOLISM\_OF\_RNA & 0.0002 & 1.4363 \\ 
  6 & REACTOME\_TRANSPORT\_OF\_SMALL\_MOLECULES & 0.0004 & 1.3724 \\ 
  7 & REACTOME\_CELLULAR\_RESPONSES\_TO\_STIMULI & 0.0004 & 1.3603 \\ 
  8 & REACTOME\_CYTOKINE\_SIGNALING\_IN\_IMMUNE\_SYSTEM & 0.0004 & 1.3561 \\ 
  9 & REACTOME\_INFECTIOUS\_DISEASE & 0.0004 & 1.3239 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0004 & 1.3083 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 23 with adjustement for confondant effect} 
\label{tab:q3_2_conf_23}
\end{table}
\subsection{Cluster 24 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0070 & 1.5747 \\ 
  2 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0044 & 1.5383 \\ 
  3 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0080 & 1.5133 \\ 
  4 & REACTOME\_O\_GLYCOSYLATION\_OF\_TSR\_DOMAIN\_CONTAINING\_PROTEINS & 0.0077 & 1.5098 \\ 
  5 & REACTOME\_RAF\_INDEPENDENT\_MAPK1\_3\_ACTIVATION & 0.0128 & 1.4582 \\ 
  6 & REACTOME\_FOXO\_MEDIATED\_TRANSCRIPTION\_OF\_OXIDATIVE\_STRESS\_METABOLIC\_AND\_NEURONAL\_GENES & 0.0122 & 1.4388 \\ 
  7 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0129 & 1.4240 \\ 
  8 & REACTOME\_SYNTHESIS\_OF\_IP3\_AND\_IP4\_IN\_THE\_CYTOSOL & 0.0136 & 1.4178 \\ 
  9 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0126 & 1.4097 \\ 
  10 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0120 & 1.4080 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 24 with adjustement for confondant effect} 
\label{tab:q3_2_conf_24}
\end{table}
\subsection{Cluster 25 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_MITOCHONDRIAL\_TRNA\_AMINOACYLATION & 0.0054 & 1.9537 \\ 
  2 & REACTOME\_RAC1\_GTPASE\_CYCLE & 0.0028 & 1.5148 \\ 
  3 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0002 & 1.4597 \\ 
  4 & REACTOME\_HIV\_LIFE\_CYCLE & 0.0054 & 1.4540 \\ 
  5 & REACTOME\_CDC42\_GTPASE\_CYCLE & 0.0054 & 1.4439 \\ 
  6 & REACTOME\_DNA\_REPAIR & 0.0013 & 1.4337 \\ 
  7 & REACTOME\_METABOLISM\_OF\_AMINO\_ACIDS\_AND\_DERIVATIVES & 0.0013 & 1.3864 \\ 
  8 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0013 & 1.3729 \\ 
  9 & REACTOME\_INTRACELLULAR\_SIGNALING\_BY\_SECOND\_MESSENGERS & 0.0036 & 1.3490 \\ 
  10 & REACTOME\_DISEASES\_OF\_SIGNAL\_TRANSDUCTION\_BY\_GROWTH\_FACTOR\_RECEPTORS\_AND\_SECOND\_MESSENGERS & 0.0030 & 1.3056 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 25 with adjustement for confondant effect} 
\label{tab:q3_2_conf_25}
\end{table}
\subsection{Cluster 26 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SCAVENGING\_OF\_HEME\_FROM\_PLASMA & 0.0020 & 1.8609 \\ 
  2 & REACTOME\_SYNTHESIS\_SECRETION\_AND\_DEACYLATION\_OF\_GHRELIN & 0.0048 & 1.7279 \\ 
  3 & REACTOME\_ROLE\_OF\_LAT2\_NTAL\_LAB\_ON\_CALCIUM\_MOBILIZATION & 0.0039 & 1.7272 \\ 
  4 & REACTOME\_CREATION\_OF\_C4\_AND\_C2\_ACTIVATORS & 0.0044 & 1.6939 \\ 
  5 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0087 & 1.6214 \\ 
  6 & REACTOME\_GASTRIN\_CREB\_SIGNALLING\_PATHWAY\_VIA\_PKC\_AND\_MAPK & 0.0090 & 1.6184 \\ 
  7 & REACTOME\_CHONDROITIN\_SULFATE\_BIOSYNTHESIS & 0.0087 & 1.5929 \\ 
  8 & REACTOME\_INITIAL\_TRIGGERING\_OF\_COMPLEMENT & 0.0075 & 1.5876 \\ 
  9 & REACTOME\_FCGR\_ACTIVATION & 0.0090 & 1.5840 \\ 
  10 & REACTOME\_CHONDROITIN\_SULFATE\_DERMATAN\_SULFATE\_METABOLISM & 0.0087 & 1.5698 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 26 with adjustement for confondant effect} 
\label{tab:q3_2_conf_26}
\end{table}
\subsection{Cluster 27 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_RNA\_POLYMERASE\_III\_TRANSCRIPTION\_TERMINATION & 0.0186 & 1.9536 \\ 
  2 & REACTOME\_NEGATIVE\_REGULATION\_OF\_NMDA\_RECEPTOR\_MEDIATED\_NEURONAL\_TRANSMISSION & 0.0223 & 1.8000 \\ 
  3 & REACTOME\_PKA\_ACTIVATION\_IN\_GLUCAGON\_SIGNALLING & 0.0223 & 1.7881 \\ 
  4 & REACTOME\_GLUTATHIONE\_CONJUGATION & 0.0223 & 1.7407 \\ 
  5 & REACTOME\_CELL\_DEATH\_SIGNALLING\_VIA\_NRAGE\_NRIF\_AND\_NADE & 0.0186 & 1.6526 \\ 
  6 & REACTOME\_INTERLEUKIN\_4\_AND\_INTERLEUKIN\_13\_SIGNALING & 0.0186 & 1.6092 \\ 
  7 & REACTOME\_NRAGE\_SIGNALS\_DEATH\_THROUGH\_JNK & 0.0223 & 1.5709 \\ 
  8 & REACTOME\_P75\_NTR\_RECEPTOR\_MEDIATED\_SIGNALLING & 0.0223 & 1.4434 \\ 
  9 & REACTOME\_RAC1\_GTPASE\_CYCLE & 0.0186 & 1.4304 \\ 
  10 & REACTOME\_DEATH\_RECEPTOR\_SIGNALLING & 0.0223 & 1.3495 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 27 with adjustement for confondant effect} 
\label{tab:q3_2_conf_27}
\end{table}
\subsection{Cluster 28 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_DETOXIFICATION\_OF\_REACTIVE\_OXYGEN\_SPECIES & 0.0029 & 1.7468 \\ 
  2 & REACTOME\_INSULIN\_RECEPTOR\_RECYCLING & 0.0089 & 1.5257 \\ 
  3 & REACTOME\_ERCC6\_CSB\_AND\_EHMT2\_G9A\_POSITIVELY\_REGULATE\_RRNA\_EXPRESSION & 0.0145 & 1.4887 \\ 
  4 & REACTOME\_DEFECTS\_OF\_CONTACT\_ACTIVATION\_SYSTEM\_CAS\_AND\_KALLIKREIN\_KININ\_SYSTEM\_KKS & 0.0145 & 1.4560 \\ 
  5 & REACTOME\_ETHANOL\_OXIDATION & 0.0104 & 1.4557 \\ 
  6 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0140 & 1.4518 \\ 
  7 & REACTOME\_SYNAPTIC\_ADHESION\_LIKE\_MOLECULES & 0.0149 & 1.4308 \\ 
  8 & REACTOME\_INTERACTION\_BETWEEN\_L1\_AND\_ANKYRINS & 0.0149 & 1.4163 \\ 
  9 & REACTOME\_RHOV\_GTPASE\_CYCLE & 0.0149 & 1.3911 \\ 
  10 & REACTOME\_NCAM\_SIGNALING\_FOR\_NEURITE\_OUT\_GROWTH & 0.0139 & 1.3894 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 28 with adjustement for confondant effect} 
\label{tab:q3_2_conf_28}
\end{table}
\subsection{Cluster 29 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_VXPX\_CARGO\_TARGETING\_TO\_CILIUM & 0.0060 & 1.6725 \\ 
  2 & REACTOME\_PTK6\_REGULATES\_RHO\_GTPASES\_RAS\_GTPASE\_AND\_MAP\_KINASES & 0.0057 & 1.6708 \\ 
  3 & REACTOME\_SYNTHESIS\_OF\_LEUKOTRIENES\_LT\_AND\_EOXINS\_EX & 0.0060 & 1.6677 \\ 
  4 & REACTOME\_PURINE\_SALVAGE & 0.0073 & 1.6352 \\ 
  5 & REACTOME\_SYNTHESIS\_OF\_PE & 0.0113 & 1.5637 \\ 
  6 & REACTOME\_EICOSANOIDS & 0.0115 & 1.5547 \\ 
  7 & REACTOME\_NUCLEOTIDE\_SALVAGE & 0.0117 & 1.5386 \\ 
  8 & REACTOME\_PHASE\_4\_RESTING\_MEMBRANE\_POTENTIAL & 0.0154 & 1.4891 \\ 
  9 & REACTOME\_EPHB\_MEDIATED\_FORWARD\_SIGNALING & 0.0130 & 1.4562 \\ 
  10 & REACTOME\_ARACHIDONIC\_ACID\_METABOLISM & 0.0123 & 1.4323 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 29 with adjustement for confondant effect} 
\label{tab:q3_2_conf_29}
\end{table}
\subsection{Cluster 30 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_SUMOYLATION\_OF\_TRANSCRIPTION\_FACTORS & 0.0217 & 1.6052 \\ 
  2 & REACTOME\_ASSEMBLY\_OF\_THE\_HIV\_VIRION & 0.0217 & 1.6038 \\ 
  3 & REACTOME\_INFLAMMASOMES & 0.0217 & 1.5898 \\ 
  4 & REACTOME\_SELENOAMINO\_ACID\_METABOLISM & 0.0194 & 1.5627 \\ 
  5 & REACTOME\_NOTCH2\_INTRACELLULAR\_DOMAIN\_REGULATES\_TRANSCRIPTION & 0.0217 & 1.5522 \\ 
  6 & REACTOME\_THE\_NLRP3\_INFLAMMASOME & 0.0223 & 1.5496 \\ 
  7 & REACTOME\_GENERATION\_OF\_SECOND\_MESSENGER\_MOLECULES & 0.0217 & 1.5421 \\ 
  8 & REACTOME\_SIGNALING\_BY\_NOTCH2 & 0.0223 & 1.4670 \\ 
  9 & REACTOME\_EUKARYOTIC\_TRANSLATION\_ELONGATION & 0.0217 & 1.4088 \\ 
  10 & REACTOME\_INFLUENZA\_INFECTION & 0.0247 & 1.2349 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 30 with adjustement for confondant effect} 
\label{tab:q3_2_conf_30}
\end{table}
\subsection{Cluster 31 }
% latex table generated in R 4.3.3 by xtable 1.8-4 package
% Thu Jun  6 18:30:41 2024
\begin{table}[H]
\centering
\scalebox{0.5}{
\begin{tabularx}{\textwidth}{rlrr}
  \hline
 & pathway & padj & NES \\ 
  \hline
1 & REACTOME\_BRANCHED\_CHAIN\_AMINO\_ACID\_CATABOLISM & 0.0010 & 1.9696 \\ 
  2 & REACTOME\_METABOLISM\_OF\_AMINO\_ACIDS\_AND\_DERIVATIVES & 0.0001 & 1.5999 \\ 
  3 & REACTOME\_SIGNALING\_BY\_INTERLEUKINS & 0.0007 & 1.4249 \\ 
  4 & REACTOME\_MEMBRANE\_TRAFFICKING & 0.0003 & 1.4035 \\ 
  5 & REACTOME\_RHO\_GTPASE\_CYCLE & 0.0010 & 1.3836 \\ 
  6 & REACTOME\_SIGNALING\_BY\_RHO\_GTPASES\_MIRO\_GTPASES\_AND\_RHOBTB3 & 0.0008 & 1.3652 \\ 
  7 & REACTOME\_METABOLISM\_OF\_RNA & 0.0007 & 1.3591 \\ 
  8 & REACTOME\_VESICLE\_MEDIATED\_TRANSPORT & 0.0003 & 1.3468 \\ 
  9 & REACTOME\_CELLULAR\_RESPONSES\_TO\_STIMULI & 0.0008 & 1.3127 \\ 
  10 & REACTOME\_INNATE\_IMMUNE\_SYSTEM & 0.0008 & 1.2648 \\ 
   \hline
\end{tabularx}
}
\caption{Top 10 up-regulated pathways for cluster 31 with adjustement for confondant effect} 
\label{tab:q3_2_conf_31}
\end{table}
\end{document}
